<!DOCTYPE html>
<html class="no-js">
<head>
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
<title></title>
<link rel="stylesheet" href="/Users/roberta.takenaka/github.com/scieloorg/packtools/packtools/catalogs/htmlgenerator/static/scielo-article-standalone.css">
<link rel="stylesheet" href="/Users/roberta.takenaka/github.com/scieloorg/packtools/packtools/catalogs/htmlgenerator/static/scielo-bundle-print.css" media="print">
<link rel="alternate" type="application/rss+xml" title="SciELO" href="">
</head>
<body class="journal article">
<a name="top"></a><div id="standalonearticle">
<section class="articleCtt"><div class="container"><div class="articleTxt">
<div class="articleBadge-editionMeta-doi-copyLink">
<span class="_articleBadge">Cell Biology</span><span class="_separator"> • </span><span class="_editionMeta">Braz J Med Biol Res 37 
                (4)
            <span class="_separator"> • </span>Apr 2004</span><span class="_separator"> • </span><span class="group-doi"><a href="https://doi.org/10.1590/S0100-879X2004000400003" class="_doi" target="_blank">https://doi.org/10.1590/S0100-879X2004000400003</a>
         
        <a class="copyLink" data-clipboard-text="https://doi.org/10.1590/S0100-879X2004000400003"><span class="sci-ico-link"></span>copy</a></span>
</div>
<h1 class="article-title">
<span class="sci-ico-openAccess showTooltip" data-toggle="tooltip" data-original-title="by 4.0 "></span>Genome features of Leptospira interrogans serovar Copenhageni<a id="shorten" href="#" class="short-link"><span class="sci-ico-link"></span></a>
</h1>
<div class="articleMeta"></div>
<div class="contribGroup">
<a href="" class="outlineFadeLink" data-toggle="modal" data-target="#ModalTutors">Authorship</a><a href="" class="outlineFadeLink" data-toggle="modal" data-target="#ModalScimago">
                SCIMAGO INSTITUTIONS RANKINGS
            </a>
</div>
<div class="row">
<ul class="col-md-2 hidden-sm articleMenu"></ul>
<article id="articleText" class="col-md-10 col-md-offset-2 col-sm-12 col-sm-offset-0"><div class="articleSection" data-anchor="Abstract"><h1 class="articleSectionTitle">Abstract</h1></div>
<div>
<p>We report novel features of the genome sequence of Leptospira interrogans serovar Copenhageni, a highly invasive spirochete. Leptospira species colonize a significant proportion of rodent populations worldwide and produce life-threatening infections in mammals. Genomic sequence analysis reveals the presence of a competent transport system with 13 families of genes encoding for major transporters including a three-member component efflux system compatible with the long-term survival of this organism. The leptospiral genome contains a broad array of genes encoding regulatory system, signal transduction and methyl-accepting chemotaxis proteins, reflecting the organism's ability to respond to diverse environmental stimuli. The identification of a complete set of genes encoding the enzymes for the cobalamin biosynthetic pathway and the novel coding genes related to lipopolysaccharide biosynthesis should bring new light to the study of Leptospira physiology. Genes related to toxins, lipoproteins and several surface-exposed proteins may facilitate a better understanding of the Leptospira pathogenesis and may serve as potential candidates for vaccine.</p>
<p>Leptospira interrogans; Outer membrane proteins; Lipopolysaccharides; Transport; Regulatory systems</p>
</div>
<hr>
<div class="articleSection" data-anchor="Text">
<p></p>
<p>
      <b>Braz J Med Biol Res, April 2004, Volume 37(4) 459-477</b>
    </p>
<p></p>
<p> Genome features of <b><i>Leptospira interrogans</i></b><b> serovar Copenhageni</b></p>
<p></p>
<p>A.L.T.O. Nascimento<sup>1</sup>, S. Verjovski-Almeida<sup>2</sup>, M.A. Van Sluys<sup>3</sup>, C.B. Monteiro-Vitorello<sup>5</sup>, L.E.A. Camargo<sup>5</sup>, L.A. Digiampietri<sup>6</sup>, R.A. Harstkeerl<sup>7</sup>, P.L. Ho<sup>1</sup>, M.V. Marques<sup>4</sup>, M.C. Oliveira<sup>3</sup>, J.C. Setubal<sup>6,10</sup>, D.A. Haake<sup>8,9</sup> and E.A.L. Martins<sup>1</sup></p>
<p></p>
<p><sup>1</sup>Centro de Biotecnologia, Instituto Butantan, São Paulo, SP, Brasil </p>
<p></p>
<p><sup>2</sup>Instituto de Química, <sup>3</sup>Instituto de Biociências and <sup>4</sup>Instituto de Ciências Biomédicas, Universidade de São Paulo, São Paulo, SP, Brasil </p>
<p></p>
<p><sup>5</sup>Escola Superior de Agricultura Luiz de Queiroz, Universidade de São Paulo, Piracicaba, SP, Brasil </p>
<p></p>
<p><sup>6</sup>Laboratório de Bioinformática, Instituto de Computação, Universidade Estadual de Campinas, Campinas, SP, Brasil </p>
<p></p>
<p><sup>7</sup>Koninklijk Instituut voor de Tropen/Royal Tropical Institute (KIT), KIT Biomedical Research, Amsterdam, The Netherlands </p>
<p></p>
<p><sup>8</sup>The David Geffen School of Medicine at University of California, Los Angeles, CA, USA </p>
<p></p>
<p><sup>9</sup>Division of Infectious Diseases, Veterans Affairs Greater Los Angeles Healthcare System, Los Angeles, CA, USA </p>
<p></p>
<p><sup>10</sup>Virginia Bioinformatics Institute and Department of Computer Science, Virginia Polytechnic Institute and State University Bioinformatics 1, Blacksburg, VA, USA </p>
<h1 class="articleSectionTitle">Abstract </h1>
<p>We report novel features of the genome sequence of <i>Leptospira interrogans</i> serovar Copenhageni, a highly invasive spirochete. <i>Leptospira</i> species colonize a significant proportion of rodent populations worldwide and produce life-threatening infections in mammals. Genomic sequence analysis reveals the presence of a competent transport system with 13 families of genes encoding for major transporters including a three-member component efflux system compatible with the long-term survival of this organism. The leptospiral genome contains a broad array of genes encoding regulatory system, signal transduction and methyl-accepting chemotaxis proteins, reflecting the organism's ability to respond to diverse environmental stimuli. The identification of a complete set of genes encoding the enzymes for the cobalamin biosynthetic pathway and the novel coding genes related to lipopolysaccharide biosynthesis should bring new light to the study of <i>Leptospira</i> physiology. Genes related to toxins, lipoproteins and several surface-exposed proteins may facilitate a better understanding of the <i>Leptospira</i> pathogenesis and may serve as potential candidates for vaccine.</p>
<p><b>Key words: </b><i>Leptospira interrogans, </i>Outer membrane proteins, Lipopolysaccharides, Transport, Regulatory systems</p>
<h1 class="articleSectionTitle">Introduction</h1>
<p>Spirochetes are motile, helically shaped bacteria which include the genera <i>Leptospira</i>, <i>Leptonema</i>, <i>Borrelia</i> and <i>Treponema</i>. <i>Borrelia</i> and <i>Treponema</i> are the causative agents of Lyme disease, relapsing fever and syphilis. <i>Leptospira</i> consists of a genetically diverse group of pathogenic and non-pathogenic or saprophytic species (1). Leptospirosis is a widespread zoonotic disease: transmission to humans occurs through contact with domestic or wild animal reservoirs or an environment contaminated by their urine. Infection produces a wide spectrum of clinical manifestations. The early phase of illness is characterized by fever, chills, headache, and severe myalgias. The disease progresses in 5 to 15% of the clinical infections to produce severe multisystem complications such as jaundice, renal failure and hemorrhagic manifestations (2). In developed countries, leptospirosis is associated with recreational activities (1) while in developing countries it produces large urban epidemics with mortality mainly during the rainy season (3). Leptospirosis also represents a major economic problem producing abortions, stillbirths, infertility, failure to thrive, reduced milk production, and death in animals such as cows, pigs, sheep, goats, horses, and dogs (1). Environmental control measures are difficult to implement because of the long-term survival of pathogenic leptospires in soil and water and the abundance of wild and domestic animal reservoirs (1). <i>Leptospira</i> are classified according to serovar status - more than 200 pathogenic serovars have been identified. Structural heterogeneity in lipopolysaccharide (LPS) moieties appears to be the basis for the large degree of antigenic variation observed among serovars (1). The development of vaccines has been pursued as a strategy for the prevention of leptospirosis. At present, vaccines are based on inactivated whole cell or membrane preparations of pathogenic leptospires which induce immune responses against leptospiral LPS (1). However, these vaccines do not induce long-term protection against infection and do not provide cross-protective immunity against heterologous leptospiral serovars. Protein antigens conserved among pathogenic serovars may contribute to overcoming the limitations of the currently available vaccines.</p>
<p> The genome sequence of <i>Leptospira interrogans</i>serovar Lai was recently published (4) and comparative genome analysis with <i>L. interrogans</i> serovar Copenhageni has been performed. We report here new features of the <i>L. interrogans</i> serovar Copenhageni that should contribute to understanding the molecular mechanisms of leptospiral physiology, pathogenesis and facilitate the identification of candidates for broad-range vaccines.</p>
<h1 class="articleSectionTitle">Material and Methods</h1>
<p>The sequenced strain, Fiocruz L1-130, was isolated as described by Nascimento et al. (5). The sequencing strategy adopted follows the basic outline of the <i>Xylella</i> genome project (6). Library construction, sequencing, assembly, and finishing were carried out by the Agronomical and Environmental Genomes consortium [<a href="http://aeg.lbi.ic.unicamp.br" target="_blank">http://aeg.lbi.ic.unicamp.br</a>] and by Instituto Butantan. The genome was assembled using phrap from shotgun reads, cosmid reads and PCR-product sequences. Scaffolding was performed using domestic software. Finishing criteria are based on consensus base phred quality of at least 20 and consensus base covered by at least one read sequence of each DNA strand (6). The first base of the sequence was chosen based on our hypothesis for the origin of the replication locus, which was in turn based on the presence of certain genes and on GC-skew variation. Genome annotation and comparative genomics were done as previously described (7). Detection of potentially surface-exposed integral membrane proteins was carried out as described by Nascimento et al. (5). Sequences from 16S rDNA were manually assembled using ESEE 3.2. Phylogenetic analyses were performed based on two matrices (34 taxa and 1255 positions; 24 taxa and 1375 positions) using the program PAUP 4.0b8 (8). Divergence time was estimated based on 1445 positions of the 16S rRNA sequences. A constant rate of 1 to 2% per 50 million years was assumed (9).</p>
<p> The sequences have been deposited in Genbank under accession numbers AE016823 (chromosome I) and AE016824 (chromosome II).</p>
<h1 class="articleSectionTitle">Results and Discussion</h1>
<p><b>Genome analysis</b></p>
<p> The <i>Leptospira</i> genome consists of two circular chromosomes with a total of 4,627,366 base pairs (bp), chromosome I with 4,277,185 bp and chromosome II with 350,181 bp (5). Circular representations of both chromosomes are depicted in <a href="" class="open-asset-modal" data-toggle="modal" data-target="#ModalFigfig1"><span class="sci-ico-fileFigure"></span>Figure 1</a>. The origin of replication of the large replicon was identified between the <i>dnaA</i> and <i>dnaN</i> genes, as in other bacterium genomes (10). GC nucleotide skew (G - C/G + C) analysis (11) confirmed the origin of replication of the large replicon and indicated two putative sites for the small replicon.</p>
<p> As previously described (12), rRNA genes in <i>L. interrogans</i> are not organized into operons, as in most other bacteria, but are scattered over the chromosome I (<a href="" class="open-asset-modal" data-toggle="modal" data-target="#ModalFigfig1"><span class="sci-ico-fileFigure"></span>Figure 1</a>). <i>L. interrogans</i> serovar Copenhageni has one <i>rrf</i> gene, two <i>rrl</i> genes and two <i>rrs</i> genes coding for 5S, 23S and 16S rRNA, respectively. As in other parasitic strains, <i>L. interrogans</i> serovar Copenhageni has only one <i>rrf </i>(5S) gene, which is located close to the origin replication region as described before for other strains of <i>L. interrogans</i> (12). Comparing the complete<i> rrs </i>(16S) sequences for <i>L. interrogans</i>, serovars Copenhageni, Lai and Canicola the identity among the sequences is of 99.9 to 100%. The <i>rrf</i> (5S) sequence identity comparing Lai and Copenhageni is 100% and the <i>rrl </i>(23S) is 99%. Based on ribosomal genes, Copenhageni and Lai are closely related, as supported by the whole genome comparison (5).</p>
<p> The Spirochaetes are divided into three major phylogenetic groups, or families: Spirochaetaceae, which includes <i>Borrelia</i> and <i>Treponema </i>among others; Brachyspiraceae and Leptospiraceae, which contain two genera, <i>Leptospira</i> and <i>Leptonema</i> (13). Phylogenetic analyses based on 16S rDNA sequences, using <i>Leptonema</i> as an outgroup, show that <i>Leptospira</i> are split into two well-supported monophyletic groups (<a href="" class="open-asset-modal" data-toggle="modal" data-target="#ModalFigfig2"><span class="sci-ico-fileFigure"></span>Figure 2</a>), one of them formed by the pathogenic strains (e.g., <i>L. interrogans</i>) and the other formed by the non-pathogenic strains (e.g., <i>L. biflexa</i>). At the base of the clade of the pathogenic strains, <i>L. inadai</i> and <i>L. fainei</i> form a well-supported assemblage. Similar results were obtained by Postic et al. (14) based on 16S rDNA analyses. In these analyses <i>L. interrogans</i> serovars formed a well-supported monophyletic cluster closely related to <i>L. kirchneri</i> (<a href="" class="open-asset-modal" data-toggle="modal" data-target="#ModalFigfig2"><span class="sci-ico-fileFigure"></span>Figure 2</a>B). Considering a constant divergence rate of 1 to 2% per 50 million years for the 16S rDNA (9), separation time between the two main assemblages (<i>L. interrogans</i> versus <i>L. biflexa</i>) was estimated at 590 to 295 million years.</p>
<p><i>L. interrogans</i> belongs to a growing number of multichromosomal prokaryotes, including <i>Vibrio cholerae</i> (15) and <i>Ralstonia solanacearum</i> (16). The small replicon of <i>L. interrogans</i> was previously suggested to be a second chromosome based upon the localization of the <i>metF</i> gene which encodes an essential biosynthetic pathway enzyme, methylene tetrahydrofolate reductase (17). The genome sequence reveals that genes encoding enzymes for metabolic pathways, such as glycolysis and the tricarboxylic acid cycle, as well as the enzymes for biosynthesis pathways of amino acid and co-factor are also distributed between the two chromosomes. Sequence analysis of chromosome II shows that an almost complete operon of genes coding for the protoheme biosynthesis pathway is present (<i>hemAIBCENYH)</i>. Although no homologue of the gene coding for uroporphyrinogen III synthetase (<i>hemD</i>) was found, experimental evidence has shown that the <i>hemC</i> gene is able to cope with <i>hemD</i> activity (18). Therefore, <i>L. interrogans</i> has the ability to synthesize protoheme<i> de novo</i>. In addition, 13 genes clustered in chromosome II coding for the cobalamin biosynthesis pathway were identified (<i>cobC</i>,<i> cobD</i>,<i> cbiP</i>,<i> cobP</i>,<i> cobB</i>,<i> cobO</i>,<i> cobM</i>,<i> cobJ</i>,<i> cbiG</i>,<i> cobI</i>,<i> cobL</i>,<i> cobH</i>,<i> cobF</i>) (<a href="" class="open-asset-modal" data-toggle="modal" data-target="#ModalFigfig3"><span class="sci-ico-fileFigure"></span>Figure 3</a>). Orthologues of<i> cobGKN</i> genes, known to be involved in the cobalamin pathway (19) were not found. However, there are two predicted coding sequences inside this operon in chromosome II that could perform these steps. One has an oxidoreductase NAD-binding domain (LIC20133) and the other is a [2Fe-2S] ferredoxin involved in electron transfer (LIC20135). In addition, other genes present in the genome coding for reductases such as LIC11145, LIC13354, LIC12391, and LIC10522 could also cope with these activities. The presence of <i>cysG</i> in chromosome I, a gene that encodes a multifunctional protein with methylase, oxidase and ferrochelatase activities (20), may also be a cobalt-inserting enzyme in the B<sub>12</sub> pathway. Other genes involved in this biosynthesis pathway were found in chromosome I (<i>cysG/hemX/cobA</i>,<i> cobT/cobU</i>,<i> cobS</i>) (<a href="" class="open-asset-modal" data-toggle="modal" data-target="#ModalFigfig3"><span class="sci-ico-fileFigure"></span>Figure 3</a>). In fact, experimental evidence has recently shown that leptospires can grow in medium in the absence of B12 (Hartskeerl RA, unpublished results). This is contrary to the previous statement that <i>L. interrogans</i> is unable to synthesize B12, and that is why it is an essential component of the EMJH semi-synthetic medium (1,4). Thus, <i>L. interrogans</i>, unlike the spirochetes <i>Borrelia burgdoferi</i> and <i>Treponema pallidum</i>, have the complete repertoire of genes for <i>de novo</i> synthesis of protoheme and cobalamin. The functional link between the two replicons supports the view that the small replicon is in fact a second chromosome.</p>
<p><div class="row fig" id="fig1">
<a name="fig1"></a><div class="col-md-4 col-sm-4"><a href="" data-toggle="modal" data-target="#ModalFigfig1"><div class="thumbOff"><div class="zoom"><span class="sci-ico-zoom"></span></div></div></a></div>
<div class="col-md-8 col-sm-8">
<strong>Figure 1.</strong><br> Circular maps of the two chromosomes in <i>Leptospira interrogans.</i> Circular representation of the <i>L. interrogans</i> serovar Copenhageni genome. The two large and small chromosomes are presented. Circles 1 and 2 (from the outside in), all predicted protein-coding regions (forward and reverse strand, respectively) color-coded by category role; circle 3: G + C content. Numbers on the outer circle are base pairs. rRNA genes, <i>rrl</i>, <i>rrs</i> and <i>rrf</i> are 23S, 16S and 5S, respectively. Note: the two replicons are not drawn to the same scale. Chromosome II (CII) is 12 times smaller than chromosome I (CI). <br>
</div>
</div></p>
<p><div class="row fig" id="fig2">
<a name="fig2"></a><div class="col-md-4 col-sm-4"><a href="" data-toggle="modal" data-target="#ModalFigfig2"><div class="thumbOff"><div class="zoom"><span class="sci-ico-zoom"></span></div></div></a></div>
<div class="col-md-8 col-sm-8">
<strong>Figure 2.</strong><br> Phylogenetic analysis of <i>Leptospira interrogans</i>. Consensus phylogenetic distance tree constructed using 16S rDNA sequences. Numbers on the branches are bootstrap values (2000 replicates) for distance (top), parsimony (below the branch) and maximum likelihood (10 replicates, below the branch, italics). GenBank accession numbers are in parenteses. <i>Leptonema</i> is used as outgroup. <i>A</i>, Based on a matrix with 34 taxa and 1255 positions; <i>B</i>, based on a matrix with 24 taxa and 1375 positions. <br>
</div>
</div></p>
<p><div class="row fig" id="fig3">
<a name="fig3"></a><div class="col-md-4 col-sm-4"><a href="" data-toggle="modal" data-target="#ModalFigfig3"><div class="thumbOff"><div class="zoom"><span class="sci-ico-zoom"></span></div></div></a></div>
<div class="col-md-8 col-sm-8">
<strong>Figure 3.</strong><br> The cobalamin biosynthetic pathway of <i>Leptospira interrogans</i>. Genetic organization of the cobalamin biosynthetic locus in the two chromosomes (CI and CII) of <i>L. interrogans</i> serovar Copenhageni. <br>
</div>
</div></p>
<p><b>Transport</b></p>
<p> Among the 220 <i>Leptospira </i>transport proteins, we found 148 proteins comprising 108 different major primary and secondary transporters (<a href="" class="open-asset-modal" data-toggle="modal" data-target="#ModalFigfig4"><span class="sci-ico-fileFigure"></span>Figure 4</a>), as defined by Paulsen et al. (21). There are 34 major primary ATP-driven transporters including 30 ABC-transporters (53 proteins), the largest protein family in <i>L. interrogans</i>, as usually is the case for bacteria (22). The ABC superfamily contains both uptake and efflux transport systems, and ATP hydrolysis energizes the transport. The porters of the ABC superfamily consist of two integral membrane proteins (that determine specificity of the transported solute) and two cytoplasmic ATP-hydrolyzing proteins present as homodimers or heterodimers. The uptake systems (but not the efflux systems) additionally possess extracytoplasmic solute-binding receptors (one or more per system), which in Gram-negative bacteria are found in the periplasm. We found 21 ABC efflux systems, including drug and heavy metal export and detoxification, lipoprotein-releasing and hemolysin export systems. The remaining 9 are ABC uptake systems, including iron, sulfate, nickel, phosphate, dipeptide, amino acid, and carbohydrate uptake. There is one F-type ATP-synthase system (8 proteins), as mentioned above, and 3 P-type cation-transport ATPases (5 proteins).</p>
<p> We found 59 secondary electrochemical potential-driven transporters (65 proteins) including the largest family of secondary transporters, the resistance-nodulation-cell division superfamily with 12 members, 5 of which belong to the heavy metal efflux family (with 7 proteins) and 7 to the multiple drug efflux family (with 8 proteins). These secondary efflux transporters are energized byproton-motive force and show the widest substrate specificityamong all known multidrug pumps, ranging from most of the currentlyused antibiotics, disinfectants, dyes, and detergents to simplesolvents. The second largest secondary transporter family is the major facilitator superfamily with 9 members including drug:cation antiporters (8 proteins) and a glycerol-phosphate:inorganic phosphate antiporter (1 protein). Additional secondary transporters include three sodium-solute symporters (one sodium-glucose co-transporter), three sodium-bile-acid symporters, and symporters for phosphate, di-tripeptides, glutamate, amino acids, uracil, sulfate as well as an ammonium:potassium antiporter, a glycerol-3-P:Pi antiporter, two drug-sodium antiporters and two polysaccharide exporters. Secondary transporters also include 7 members of the TonB family of auxiliary proteins for energizing the outer membrane receptor-mediated active transport. <i>Leptospira</i> has many iron-transporting proteins in addition to the cobalamin/iron periplasmic binding protein component of the ABC-transport system (LIC13403) mentioned above. Four members of the outer membrane receptor family have been identified, which are probably involved in the transport of iron dicitrate (LIC10714), hemin (LIC10964), ferric hydroxamate (LIC11345) and ferrienterobactin (LIC12998). In addition, an orthologue of the ferrous iron up- take (FeoB) protein of <i>E. coli</i> was found (LIC11402). FeoB has been characterized as an Fe<sup>2+</sup> uptake system and possesses an ATP/GTP binding motif at its N-terminal hydrophilic domain, therefore being probably energized by ATP or GTP hydrolysis (23).</p>
<p><div class="row fig" id="fig4">
<a name="fig4"></a><div class="col-md-4 col-sm-4"><a href="" data-toggle="modal" data-target="#ModalFigfig4"><div class="thumbOff"><div class="zoom"><span class="sci-ico-zoom"></span></div></div></a></div>
<div class="col-md-8 col-sm-8">
<strong>Figure 4.</strong><br> Major transporters of <i>Leptospira interrogans</i>. Major primary transporters are driven by ATP hydrolysis and include ABC-transporters, P-type ion transporters and the ATP-synthase. All other transporters indicated are the major secondary transporters, which are driven by the transmembrane electrochemical potential. RND = resistance-nodulation-cell division superfamily. <br>
</div>
</div></p>
<p><b>Oxygen defense</b></p>
<p> The <i>Leptospira </i>genome contains several genes encoding enzymes with peroxidase activities, such as catalase, glutathione peroxidase and thiol peroxidase. However, superoxide dismutase orthologues and two important regulons, SoxRS and OxyR, normally responsible for the defense against oxidative stress, are absent (5). It is possible that metalloporphyrins (24) could provide defense against oxidative damage, since <i>L. interrogans</i> is competent for porphyrin biosynthesis and has metal uptake transport proteins. Genes coding for co-migratory bacterioferritin (LIC20093 and LIC10732), thiol peroxidase (LIC12765) and peroxiredoxin (LIC11219) with alkyl hydroperoxide reductase (AhpC) and thiol-specific antioxidant activities, respectively, were also identified. Two predicted coding sequences for bacterioferritin (LIC11310 and LIC13209), which may have functions analogous to animal ferritin, are also present and may provide both iron detoxification and storage, that would prevent free iron in <i>Leptospira</i> from driving oxidative reactions.</p>
<p><b>Regulatory systems and signal transduction</b></p>
<p> There are many genes encoding signal transduction proteins in <i>Leptospira</i>, indicating that this organism has developed a vast array of regulatory systems that enable it to respond to environmental signals. This variety of regulatory domains is found in non-obligate parasitic bacteria, indicating a much greater need to interpret the signals from the environment in order to respond properly, while obligate parasites have a much lower number and variety of domains in their signal transduction proteins (25). There are 80 genes encoding components of the phosphorylation-mediated signal transduction pathway: 29 histidine kinases (HK), 30 response regulators (RR) and 18 hybrid kinase/regulators (HK/RR). These members of the two-component systems present several domains organized into different arrangements (<a href="" class="open-asset-modal" data-toggle="modal" data-target="#ModalFigfig5"><span class="sci-ico-fileFigure"></span>Figure 5</a>). Nineteen of the histidine kinases are located in the inner membrane, nine are cytoplasmic and one is probably found in the periplasm, as predicted by the PSORT program [<a href="http://psort.nibb.ac.jp/" target="_blank">http://psort.nibb.ac.jp/</a>].</p>
<p> On the other hand, only a third of the hybrid HK/RR are membrane-bound, suggesting that these hybrid proteins are most likely to be involved in phosphorylation cascades, although some of them have the sensor PAS domain. The PAS domain has been reported to sense different environmental stimuli, like oxygen, redox state, nitrogen availabilityor light, and it may or may not be associated with co-factors such as heme or FAD (26). A two-component protein containing a PAS sensor domain was found to be required for virulence of <i>M. tuberculosis</i> in mice, probably because it senses radical oxygen intermediates generated by macrophage phagocytosis (27), and probably this is also the case for <i>Leptospira</i>.</p>
<p> The response regulators are the cytoplasmic effectors of the message, which become functional after being phosphorylated at an aspartate residue by the cognate histidine kinase. The RRs may possess a second effector domain (<a href="" class="open-asset-modal" data-toggle="modal" data-target="#ModalFigfig5"><span class="sci-ico-fileFigure"></span>Figure 5</a>), which will perform its ultimate function, such as the DNA-binding helix-turn-helix domain that allows the RR to regulate transcription. Other noticeable domains found in <i>L. interrogans</i> RRs are the GGDEF and EAL motifs, which correspond to putative diguanylate cyclase and phosphodiesterase domains, respectively, and a phosphatase domain similar to the mammalian phosphatase 2C that may be involved in the phosphorelay.</p>
<p> Cyclic nucleotides are likely to have a major regulatory role in <i>Leptospira</i>. There are 19 homologues of adenylate/guanylate cyclases, two specific phosphodiesterases and 7 cyclic nucleotide-binding proteins that probably have a regulatory function. There are 12 genes encoding proteins containing the GGDEF motif, seven of which are organized in tandem in chromosome I (LIC11131 to LIC11125), and they also have a PAS/PAC sensor domain in the amino-terminal region, suggesting that they are the product of gene duplication. The diguanylate cyclase activity of the GGDEF domain is required for a novel regulatory mechanism involving bis-(2',5')-cyclic diguanylic acid (c-di-GMP) as an allosteric activator (28). Two of the GGDEF-containing proteins also have the Cache domain, which is involved in chemotaxis signal transduction (29), an important feature for <i>Leptospira</i>. There are eight putative diguanylate phosphodiesterases containing an EAL domain, with three of them being associated with an RR domain (<a href="" class="open-asset-modal" data-toggle="modal" data-target="#ModalFigfig5"><span class="sci-ico-fileFigure"></span>Figure 5</a>). <i>L. interrogans</i> also presents in its genome three serine/threonine kinases and two hybrid HK/RR with a GAF domain, which is a binding domain for cGMP (30).</p>
<p> Other interesting findings in the genome include three related genes encoding putative DNA-binding proteins (LIC20104, LIC20105 and LIC20178), which contain 6 transmembrane domains in the amino-terminus and one helix-turn-helix domain of the AraC type at the carboxyl-terminus. These genes are present in chromosome II, and two of them are clustered. Orthologues with a C-terminal DNA-2binding domain and a hydrophobic N-terminal region were described in other bacteria, including <i>Borrelia</i> and <i>Treponema</i> (31), but their function is unknown.</p>
<p><div class="row fig" id="fig5">
<a name="fig5"></a><div class="col-md-4 col-sm-4"><a href="" data-toggle="modal" data-target="#ModalFigfig5"><div class="thumbOff"><div class="zoom"><span class="sci-ico-zoom"></span></div></div></a></div>
<div class="col-md-8 col-sm-8">
<strong>Figure 5.</strong><br> Domain architecture of putative signal transduction proteins from <i>Leptospira interrogans.</i> The domain organization of histidine kinases, response regulators, hybrid histidine kinase/response regulators and other peptides probably associated with signal transduction is shown schematically. The number of peptides containing each domain organization is shown on the right, and the genes are found distributed over the large or small replicons, except for the seven peptides containing the PAS/GGDEF arrangement, which are encoded by genes located in tandem in the large replicon. AAA = ATPase; Cache = small ligand binding domain; EAL = phosphodiesterase; GAF = cGMP-binding domain; GGDEF = diguanylate cyclase; HK = histidine kinase domain; HTH = helix-turn-helix; PAS = sensor domain (may include a PAS/PAC arrangement); PP = phosphatase; RR = response regulator domain; SP = signal peptide; TM = transmembrane region. <br>
</div>
</div></p>
<p><b>Motility and chemotaxis</b></p>
<p> The <i>L. interrogans</i> genome comprises a relatively large number of motility and chemotaxis genes. Enteric bacteria usually have about 50 genes coding for structural and functional proteins involved in motility (32). A similar number of genes have been identified for the spirochetes <i>T. pallidum</i> and <i>B. burgdorferi</i> (33,34). Apparently, the motility and chemotaxis apparatus of <i>L. interrogans</i> is more complex since its genome contains at least 79 putative motility-associated genes. All genes are well conserved among <i>L. interrogans</i>, <i>T. pallidum</i> and <i>B. burgdorferi</i> and 42 genes were found to be common to all three genera. However, the leptospiral genome contains multiple copies of a number of motility-associated genes, accounting partly for the higher number. For example, the genome of serovar Copenhageni contains 5 <i>flaB</i> genes, 4 <i>motB</i> genes and 2 <i>motA</i> genes compared to 3 <i>flaB</i> genes and one copy of both <i>motA</i> and <i>motB</i> in the <i>T. pallidum</i> genome and one copy each of <i>flaB</i>, <i>motA</i>, and <i>motB</i> in the <i>B. burgdorferi</i> genome. In addition, the <i>L. interrogans</i> genome contains 11 putative genes encoding methyl-accepting chemotaxis proteins, which is roughly twice as many as in <i>T. pallidum</i> and <i>B. burgdorferi</i>. Forty-eight of the 79 motility-associated genes are positioned into 14 gene clusters varying in size from 2 to 8 genes. Thus, like in <i>T. pallidum</i> and <i>B. burgdorferi</i>, the majority of the structural and functional motility genes are positioned in potential operons. However, the operons probably underwent extensive rearrangements as they are generally smaller, often corresponding to only parts of the major <i>Treponema</i> and <i>Borrelia</i> operons. For example, the <i>flgB </i>operon in <i>B. burgdorferi</i> consisting of 26 genes (35) has been disrupted into 6 fragments dispersed at 5 positions in the leptospiral genome. Differences in number of genes and operon organization might be associated with the high flexibility of pathogenic leptospires enabling them to survive and adapt to a variety of environments and hosts.</p>
<p><b>Hemolysins</b></p>
<p> The primary lesion caused by <i>Leptospira</i> is the damage to the endothelium of small blood vessels, leading to hemorrhage and localized ischemia in multiple organs. As a consequence, renal tubular necrosis, hepatocellular damage, meningitis, and myositis may occur in the infected host (1). Hemolysins may play a fundamental role in this toxic process. Several genes coding for predicted hemolysins were identified in the <i>L. interrogans</i> genome. Some of them are structurally related to sphingomyelinases C (LIC10657, LIC12631, LIC12632, LIC11040, and LIC13198). Although generically called sphingomyelinases, it is possible that these genes code for hydrolases that act not only on sphingomyelins but also on other sphingolipids. Erythrocytes may represent a target for these enzymes since they are rich in glycosphingolipids like the antigenic determinants of the ABO system. Interestingly, LIC12631 and LIC12632 are organized as an operon, which may suggest a concerted expression and action.</p>
<p> The other class of genes coding for hemolysins, <i>tlyABC</i>, was identified. Although they were assigned to the same TlyABC class, they do not present structural similarity. These putative hemolysins were first identified in the spirochete <i>Brachyspira hyodysenteriae</i>. Hemolytic and cytotoxic activities of the recombinant TlyA, TlyB and TlyC proteins, expressed in <i>E. coli</i>, were detected (36). TlyB belongs to the family of Clp ATP-dependent proteases (37) and there are 3 genes coding for structurally related proteins (LIC10339, LIC11814 and LIC12017). Thus, 5 genes of the <i>tlyABC</i> class (LIC10284, LIC10339, LIC11814, LIC12017, and LIC13143) were identified in the <i>L. interrogans</i> genome.</p>
<p> LIC12134 codes for an HlpA-related protein which was characterized as a hemolysin in <i>Prevotella intermedia</i>, a common oral bacterium associated with periodontitis (38). Another identified gene (LIC10325) is related to the <i>hlyX</i> gene which codes for a predicted hemolysin previously identified in <i>L. borgpetersenii</i> serovar Hardjo (Accession number AAF09252, unpublished results). LIC11352 should also be mentioned as the gene which codes for LipL32 or HAP-1, a ubiquitous lipoprotein of pathogenic <i>Leptospira</i> (see Lipoproteins section below) with hemolytic activity (39).</p>
<p><b>Surface-exposed proteins</b></p>
<p> Pathogenic leptospires require several types of surface-exposed proteins for the purpose of colonization and survival in the mammalian host. Surface-exposed proteins may be categorized as nonspecific porins, specific channels for nutrient acquisition, efflux channels, lipoproteins, adhesins, S-layer glycoproteins, peripheral membrane proteins, or surface-maintenance proteins. Aside from S-layer proteins and peripheral membrane proteins, these surface-exposed proteins would be expected to be integrated into the outer membrane via transmembrane regions or lipid modification. The leptospiral genome contains homologues of SecY and other secretory proteins involved in exporting proteins with signal peptides across the cytoplasmic membrane. The leptospiral genome also contains both the standard signal peptidase and the lipoprotein signal peptidase. The standard signal peptidase is involved in the hydrolysis of signal peptides of transmembrane outer membrane proteins and periplasmic proteins, releasing them from the cytoplasmic membrane. Lipoprotein signal peptidase hydrolyzes the signal peptides of lipoproteins prior to lipidation of the N-terminal cysteine.</p>
<p><b>Transmembrane outer membrane proteins</b></p>
<p> Analysis of the <i>L. interrogans</i> genome reveals 83 beta-sheet transmembrane outer membrane proteins (OMPs) and all except one (OmpL1, LIC10973) (40) were previously unknown. An example of a newly identified leptospiral protein with a transmembrane beta-sheet structure is LIC10642 which is predicted to be a member of the OMP superfamily.</p>
<p> Among these newly identified OMPs is a family of predicted coding sequences (LIC20202, LIC10995, LIC11465, and LIC11103) belonging to the superfamily of alpha/beta hydrolases, which includes bacterial lipases. Another transmembrane OMP is LIC11623, a member of a family of highly conserved proteins of Gram-negative bacteria, including <i>N. meningitides</i> Omp85, which is thought to be a chaperonin involved in the assembly of OMPs in the outer membrane (41). OmpL1 belongs to the class of nonspecific porins allowing passage of small molecules (&lt;1000 Daltons) across the leptospiral outer membrane (40). Nonspecific porins allow both influx of nutrients and efflux of products of metabolism. Another example is LIC11458, which is predicted to be a member of the porin superfamily.</p>
<p><b>Specific channels</b></p>
<p> Survival in the mammalian host environment by bacterial pathogens requires the acquisition of certain trace nutrients. For example, iron is essential for leptospiral growth and is bound by a number of high-affinity binding proteins in mammals, which restrict its availability. Bacteria scavenge trace nutrients, including iron, heme, and vitamin B12, from their environment utilizing the cytoplasmic membrane protein TonB and a series of "TonB-dependent" OMPs (<a href="" class="open-asset-modal" data-toggle="modal" data-target="#ModalFigfig6"><span class="sci-ico-fileFigure"></span>Figure 6</a>). After binding, transport of the nutrient across the outer membrane into the periplasm by this type of channel is an energy-dependent step requiring interaction of the TonB-dependent OMP with TonB. The <i>L. interrogans</i> genome contains a TonB orthologue (LIC10889) and a large number of TonB-dependent OMPs: LIC20151, LIC20214, LIC10998, LIC10964, LIC10714, LIC12374, LIC12898, LIC12998, LIC11694, LIC11345, LIC10882, LIC10881, and LIC10896.</p>
<p> The leptospiral genome also contains OMPs involved in efflux pathways (<a href="" class="open-asset-modal" data-toggle="modal" data-target="#ModalFigfig6"><span class="sci-ico-fileFigure"></span>Figure 6</a>). The type I efflux system is represented </p>
<p> by an orthologue of TolC (LIC13135), the outer membrane exporter of hemolysin and drugs, along with an orthologue of CusC (LIC11941), an outer membrane exporter of copper ion. A second type of efflux pathway found in the leptospiral genome belongs to the resistance-nodulation-cell division superfamily, a three-member complex that catalyzes substrate efflux via an H<sup>+</sup> antiport mechanism. The three-member complex consists of an resistance-nodulation-cell division transporter, a membrane fusion protein, and an outer membrane factor involved in the export of proteins, carbohydrates, drugs or toxic heavy metals (<a href="" class="open-asset-modal" data-toggle="modal" data-target="#ModalFigfig6"><span class="sci-ico-fileFigure"></span>Figure 6</a>). The leptospiral orthologues are most closely related to those of the <i>Alcaligenes eutrophus</i> cobalt/cadmium/zinc export system consisting of the resistance-nodulation-cell division transporter CzcA (LIC12224, LIC15510 and LIC11938), CzcB (LIC12306 and LIC11940), and CzcC (LIC12307 and LIC11941). In addition, <i>L. interrogans</i> has two orthologues of the cation efflux system protein CzcD (LIC11714 and LIC13205), which are members of the cation diffusion facilitator family. CzcD of <i>Bacillus subtilis</i> has been shown to catalyze divalent cation (Zn<sup>2+</sup> or Cd<sup>2+</sup>) efflux in exchange for the uptake of two monovalent cations (K<sup>+</sup> and H<sup>+</sup>) in an electroneutral process energized by the transmembrane pH gradient (42). </p>
<p><div class="row fig" id="fig6">
<a name="fig6"></a><div class="col-md-4 col-sm-4"><a href="" data-toggle="modal" data-target="#ModalFigfig6"><div class="thumbOff"><div class="zoom"><span class="sci-ico-zoom"></span></div></div></a></div>
<div class="col-md-8 col-sm-8">
<strong>Figure 6.</strong><br> Model of leptospiral membrane architecture. Leptospires have two membranes, an outer membrane (OM) and a cytoplasmic or inner membrane (IM). As in Gram-positive bacteria, the peptidoglycan (PG) cell wall is closely associated with the IM. The leptospiral surface is dominated by lipopolysaccharide (LPS) carbohydrate side chains. Subsurface proteins include the cytoplasmic protein, GroEL, and the periplasmic endoflagella (EF). The IM contains lipoproteins such as LipL31 and transmembrane proteins such as signal peptidase (SP) and ImpL63. The OM contains lipoproteins including LipL41 and LipL36, and transmembrane proteins including the porin, OmpL1. Genomic sequence analysis reveals several novel types of outer membrane proteins (OMPs), including TonB-dependent OMPs involved in nutrient acquisition. BtuB is an example of a TonB-dependent OMP involved in uptake of vitamin B<sub>12</sub>. The type I efflux system is represented by TolC, which forms a complex with ATP-binding cassette (ABC) transporters to export hemolysins and other cytoplasmic components. The leptospiral genome also contains genes involved in a three-component efflux system consisting of an outer membrane factor (OMF), membrane fusion protein (MFP), and an inner membrane transporter, in this case CzcA, which is involved in heavy metal detoxification. <br>
</div>
</div></p>
<p><b>Lipoproteins</b></p>
<p> Experimental evidence for fatty acid modification of leptospiral lipoproteins has been described for the outer membrane lipoproteins LipL32 (LIC11352) (43), LipL36 (LIC13060) (44), and LipL41 (LIC12966) (45). The cytoplasmic membrane also contains lipoproteins, as demonstrated for LipL31 (LIC11456) and LipL71 (LIC11003) (46). A total of 184 predicted coding sequences in the <i>L. interrogans</i> genome were found to have a lipoprotein signal peptidase cleavage site (5). All proposed lipoprotein-coding sequences conform to the rule of having an L, I, V, or F in the -3 and/or -4 position relative to cysteine and most of them have A, G, S, or N in the -1 position relative to cysteine (47).</p>
<p> Proteolytic functions can be assigned to some of the newly identified lipoproteins: four lipoproteins are thermolysin homologues (LIC10715, LIC13320, LIC13321, and LIC13322), and one is a predicted metalloprotease (LIC11834). Several lipoproteins may be involved in hemolysis: two lipoproteins are sphingomyelinase homologues (LIC10657 and LIC12632), and one is a phospholipase D homologue (LIC11754). Lipoprotein LIC10972 is predicted to be located in the outer membrane and is a MauG homologue belonging to a family of cytochrome c peroxidases that may be involved in defense against hydrogen peroxide. Other lipoproteins with putative enzymatic functions are homologues of glucose dehydrogenase (LIC12294) and cholesterol oxidase (LIC13202).</p>
<p><b>S-layer and peripheral membrane proteins</b></p>
<p> S-layers are thought to provide structural integrity to the bacterial surface (48). Although an S-layer has not been observed in pathogenic leptospires, the genome contains at least two proteins with S-layer homology (LIC10868 and LIC12952). Experimental evidence is needed to determine whether these proteins are actually S-layer components.</p>
<p> Like S-layer proteins, peripheral membrane proteins are not integral membrane proteins. LipL45 is processed to a peripheral membrane associated with the outer membrane, P31<sub>LipL45</sub> (49). P31<sub>LipL45</sub> expression is dramatically up-regulated in stationary phase cultures, and for this reason may have a membrane-stabilizing function. At this time it is unclear whether P31<sub>LipL45</sub> is surface-exposed. Interestingly, the leptospiral genome contains a number of predicted coding sequences with homology to LipL45 (LIC20102, LIC20114, LIC13414, and LIC10123).</p>
<p><b>Surface-maintenance proteins</b></p>
<p> Bacteria are likely to have a variety of proteins involved in maintaining the surface structural integrity. One such protein, glycerophosphoryl diester phosphodiesterase, is a protein belonging to this category which has been found in all spirochetal genomes studied to date. <i>E. coli </i>has two forms of this enzyme, a cytosolic form, ugpQ, and a periplasmic form, glpQ. The leptospiral genome contains two homologues of this enzyme, LIC13182 and LIC10293. The former should be the cytosolic form because it lacks a signal peptide while the latter should be the exported form because it has an N-terminal signal peptide. In other spirochetes, GlpQ is associated with the outer membrane (50).</p>
<p><b>Cytoplasmic membrane proteins</b></p>
<p> The leptospiral genome contains a number of proteins that belong to a large family of prokaryotic proteins with homology to the peptidoglycan-associated portion of <i>E. coli</i> OmpA (51). These proteins are predicted to be either cytoplasmic membrane (LIC20250, LIC10592, LIC13479, and LIC10050) or periplasmic proteins (LIC10537 and LIC10191), rather than OMPs, which is consistent with experimental evidence that the spirochetal cell wall is more closely associated with the cytoplasmic membrane than the outer membrane. An interesting protein family is the mechanosensitive ion channel (LIC20069 and LIC12671). Two members of this protein family of <i>M. jannaschii</i> have been functionally characterized and both form mechanosensitive ion channels (52). Therefore, this family is likely to also encode mechanosensitive channel proteins in <i>L. interrogans</i>, playing a physiological role in bacterial osmoregulation.</p>
<p><b>Lipopolysaccharides</b></p>
<p> Lipopolysaccharides distinguish the leptospiral surface from those of the other invasive spirochetes. Changes in genes involved in the LPS biosynthesis apparatus are thought to account for serovar diversity among leptospires (53). It has been shown that the leptospiral LPS resembles a typical LPS from Gram-negative bacteria, containing pentoses, hexoses and heptoses. Although the predominant sugar is rhamnose, a large variety of other sugar residues are found (1). Antibodies raised against LPS from different <i>Leptospira</i> strains during infections are related to polysaccharide structure in terms of its sugar composition, number, repetitiveness, and ramification (1). In <i>Leptospira</i>, as in many other bacteria, at least part of the genes coding for enzymes of the polysaccharide biosynthesis pathway are found clustered in a region of chromosomes named O-antigen gene cluster (<i>rfb</i> locus) (53).</p>
<p> The complete genome analysis reveals a large portion of about 119 kb (genome position 2.538.470-2.554.255), in which all the 108 predicted coding sequences are transcribed in the same orientation (<a href="" class="open-asset-modal" data-toggle="modal" data-target="#ModalFigfig7"><span class="sci-ico-fileFigure"></span>Figure 7</a>). Interestingly, this region is dense in genes related to LPS biosynthesis (<a href="" class="open-asset-modal" data-toggle="modal" data-target="#ModalTablet01"><span class="sci-ico-fileTable"></span>Table 1</a>) and includes the O-antigen gene cluster previously described in <i>L. interrogans </i>serovar Copenhageni (53,54). In the first 14 kb of this region the predicted genes seem not to be related to LPS biosynthesis, but to DNA replication, and some genes code for ribosomal proteins. In the other 105 kb of this region there are 94 predicted genes 56 of which are clearly related to LPS biosynthesis. These predicted genes encode enzymes for nucleotide sugar biosynthesis, such as the enzymes for dTDP-rhamnose biosynthesis (54) for CMP-N-acetylneuraminic acid and for perosamine synthase. In addition, many genes coding for sugar transferases, sugar modifications and for the O-antigen processing, Wzx-flippase and Wzy-polymerase, involved in oligosaccharide exporting and assembly of the LPS, were identified. Some genes encoding proteins of the lipid A biosynthesis (<i>lpxD</i>) and transport (<i>msbA</i>) are also found is this region.</p>
<p><div class="row table" id="t01">
<a name="t01"></a><div class="col-md-4 col-sm-4"><a data-toggle="modal" data-target="#ModalTablet01"><div class="thumbOff">
                        Thumbnail
                        <div class="zoom"><span class="sci-ico-zoom"></span></div>
</div></a></div>
<div class="col-md-8 col-sm-8"><br></div>
</div></p>
<p>
 Genome features of <i>Leptospira interrogans</i> serovar Copenhageni. <i>A.L.T.O. Nascimento, S. Verjovski-Almeida, M.A. Van Sluys, C.B. Monteiro-Vitorello, L.E.A. Camargo, L.A. Digiampietri, R.A. Harstkeerl, P.L. Ho, M.V. Marques, M.C. Oliveira, J.C. Setubal, D.A. Haake and E.A.L. Martins</i>. Brazilian Journal of Medical and Biological Research, 37 (4): 459, 2004. </p>  <p> The comparison with the corresponding region in the genome of <i>L. interrogans</i> serovar Lai (4) revealed only three distinct predicted coding sequences: two extra copies of genes for aminotransferase (LIC12197 and LIC12198) and the absence of a gene encoding galactoside O-acetyltransferase (LA1622).</p> <p> Seventy-seven other genes, probably related to LPS biosynthesis, were identified along the genome. All the genes related to lipid A biosynthesis described in <i>E. coli</i> were identified in the <i>Leptospira</i> genome (<i>lpxA</i>, <i>lpxC</i>, <i>lpxD</i>, <i>lpxB</i>, <i>lpxK</i>). However, the predominant fatty acids in lipid A of <i>L. interrogans</i> are dodecanoic and hexadecanoic acid instead of hydroxymyrystoyl (14 carbons), which is the signature of Gram-negative bacteria (1). Comparative analysis between LpxA of <i>E. coli</i> and <i>P. aeruginosa</i> (55) showed that few structural differences in this enzyme determine changes in the fatty acid chain size incorporated during lipid A biosynthesis. The lower endotoxicity of leptospiral LPS as compared to LPS from Gram-negative bacteria has been reported (56). It was also reported that leptospiral LPS induces a TLR2-dependent cell activation, instead of LTR4, the receptor frequently involved in the LPS immune response (57). The lower endotoxicity and the difference in the mechanism of cell immune response activation are supposed to be related to differences in the lipid A structure.</p> <p> Genes encoding enzymes of Kdo biosynthesis (<i>kdsA</i>, <i>kdsB</i>) such as Kdo-transferase (<i>waaA </i>or <i>kdtA</i>), which catalyzes the binding of Kdo to lipid IVA, were identified. Although typical Kdo was found in <i>Leptospira</i> LPS, it was reported to be substituted at different carbon positions (1).</p> <p> Four paralogues encoding MsbA, the protein that transfers the flippase of lipid A in the inner membrane from the cytoplasmic side to the periplasmic side, were identified. One of the <i>msbA</i> genes is located in the 119-kb region. Another <i>msbA</i> gene is located upstream of <i>lpxK</i>, as reported in many other genomes (55). It will be interesting to compare the set of genes coding for enzymes related to LPS biosynthesis identified in the <i>Leptospira</i> genome to the orthologues in other microorganisms, in order to correlate LPS structural differences and endotoxic activity.</p>    <p><div class="row fig" id="fig7">
<a name="fig7"></a><div class="col-md-4 col-sm-4"><a href="" data-toggle="modal" data-target="#ModalFigfig7"><div class="thumbOff"><div class="zoom"><span class="sci-ico-zoom"></span></div></div></a></div>
<div class="col-md-8 col-sm-8">
<strong>Figure 7.</strong><br> Lipopolysaccharide region of <i>Leptospira interrogans.</i> The 119-kb region of <i>L. interrogans</i> serovar Copenhageni genome with 108 predicted coding sequences transcribed in the same orientation. Orange arrows indicate predicted genes encoding polysaccharide biosynthesis-related proteins. Shaded regions indicate the homologous genes from <i>L. borgpetersenii</i> serovar Hardjobovis (53). The region containing the predicted coding sequences, indicated by the letters <i>F, E, A, B, D, C</i>, was first indicated as the <i>rfb</i> genes related to rhamnose biosynthesis in <i>L. interrogans</i> Copenhageni (54). <br>
</div>
</div></p> <p><b>Comparative genomics and insertion sequences</b></p> <p> A three-way comparison between the<i> L. interrogans</i> genome and the genomes of <i>B.</i> <i>burgdorferi</i> and <i>T. pallidum</i> yields the following results: 1167 (31%) of the genes in <i>L. interrogans</i> Copenhageni are found in <i>B. burgdorferi </i>and/or in <i>T. pallidum</i>, 666 (41%) of the genes in <i>B. burgdorferi </i>are found in the Copenhageni genome, and 589 (57%) of the genes in <i>T. pallidum </i>are found in the <i>L. interrogans</i> genome. A total of 362 predicted genes were found to be shared by all three spirochetes, 45 of which are hypothetical (detailed list in our project website: <a href="http://aeg.lbi.ic.unicamp.br/world/lic/" target="_blank">http://aeg.lbi.ic.unicamp.br/world/lic/</a>). These results show that a thorough analysis of the genome can significantly contribute to the understanding of spirochete biology.</p> <p> Our analysis of the <i>L. interrogans</i> genome revealed the presence of the previously described insertion sequence (IS) elements IS<i>1500</i>, IS<i>1501</i> (58), reminiscent insertions of IS<i>1533</i> (59) and IS<i>1502</i> (60)and an IS recently identified that we designated IS<i>lin1</i> (5) (<a href="" class="open-asset-modal" data-toggle="modal" data-target="#ModalTablet02"><span class="sci-ico-fileTable"></span>Table 2</a>). IS<i>1500</i>, IS<i>1501</i> and IS<i>lin1</i> account for 24 insertions distributed throughout chromosome I. Together, IS elements and transposases comprise 2% of the genome. These elements are related to major IS families such as IS110 and IS3 that are defined by their conserved amino acid motif (DDE) in the transposase. So far all the insertions were found in intergenic regions, having no mutagenic effects on <i>L. interrogans</i> genes.</p>
<p><div class="row table" id="t02">
<a name="t02"></a><div class="col-md-4 col-sm-4"><a data-toggle="modal" data-target="#ModalTablet02"><div class="thumbOff">
                        Thumbnail
                        <div class="zoom"><span class="sci-ico-zoom"></span></div>
</div></a></div>
<div class="col-md-8 col-sm-8"><br></div>
</div></p>

 <p>
 Genome features of <i>Leptospira interrogans</i> serovar Copenhageni. <i>A.L.T.O. Nascimento, S. Verjovski-Almeida, M.A. Van Sluys, C.B. Monteiro-Vitorello, L.E.A. Camargo, L.A. Digiampietri, R.A. Harstkeerl, P.L. Ho, M.V. Marques, M.C. Oliveira, J.C. Setubal, D.A. Haake and E.A.L. Martins</i>. Brazilian Journal of Medical and Biological Research, 37 (4): 459, 2004. </p>  <p> The <i>L. interrogans</i> genome provides new insights into biosynthesis pathway, transport families, environmental response, and pathogenesis. A broad array of regulatory system proteins enable this organism to respond to environmental signals. A new group of genes involved in LPS biosynthesis may contribute to the elucidation of serovar diversity among leptospires. Several categories of surface-exposed proteins required for colonization and survival in the mammalian host were identified. These proteins may have a role in mechanisms of leptospiral pathogenesis and protective immunity. Available vaccines are serovar-specific and have low efficacy (1). Surface-exposed proteins that are conserved among pathogenic serovars may be used for vaccine development for the prevention of leptospirosis.</p>   <p><b>Legend of insert that is included in the print version of the Brazilian Journal of Medical and Biological Research</b></p> 


<p>
    <a target="_blank" href="1414-431X-bjmbr-37-04-459-477-encarte_screen.pdf"><img style="max-width:100%" src="1414-431X-bjmbr-37-04-459-477-gleptospirapeq.gif">
        Insert (included in the print version of the Brazilian Journal of Medical and Biological Research).
    
    Linear representation of the Leptospira interrogans serovar Copenhageni chromosomes (CI and CII). Genes are colored according to their biological role. Arrows indicate the direction of transcription. The pie representation indicates the distribution of the number of genes according to their biological role. The numbers below protein-producing genes correspond to gene identification numbers (IDs). 
</a>
</p>

<p> Additional information and insert is contained in the project website <a href="http://aeg.lbi.ic.unicamp.br/world/lic/" target="_blank">http://aeg.lbi.ic.unicamp.br/world/lic/</a>
    </p>
</div>
<div class="articleSection" data-anchor="References">
<h1 class="articleSectionTitle">References</h1>
<div class="ref-list"><ul class="refList">
<li><div>1. Faine S, Adler B, Bolin C &amp; Perolat P (1999). <i>Leptospira and Leptospirosis</i>    MediSci, Melbourne, Australia.</div></li>
<li><div>2. Bharti AR, Nally JE, Ricaldi JN et al. (2003). Leptospirosis: a zoonotic    disease of global importance. <i>Lancet Infectious Diseases</i>, 3: 757-771.</div></li>
<li><div>3. Romero EC, Bernardo CC &amp; Yasuda PH (2003). Human leptospirosis: a twenty-nine-year    serological study in São Paulo, Brazil. <i>Revista do Instituto de Medicina    Tropical de São Paulo</i>, 45: 245-248.</div></li>
<li><div>4. Ren SX, Fu G, Jiang XG et al. (2003). Unique physiological and pathogenic    features of <i>Leptospira interrogans</i> revealed by whole-genome sequencing.    <i>Nature</i>, 422: 888-893.</div></li>
<li><div>5. Nascimento ALTO, Ko AI, Martins EAL et al. (2004). Comparative genomics    of two <i>Leptospira interrogans </i>serovars<i> </i>reveals novel insights    into physiology and pathogenesis. <i>Journal of Bacteriology</i> (in press).</div></li>
<li><div>6. Simpson AJ, Reinach FC, Arruda P et al. (2000). The genome sequence of    the plant pathogen <i>Xylella fastidiosa</i> <i>Nature</i>, 406: 151-157.</div></li>
<li><div>7. Van Sluys MA, de Oliveira MC, Monteiro-Vitorello CB et al. (2003). Comparative    analyses of the complete genome sequences of Pierce's disease and citrus variegated    chlorosis strains of <i>Xylella fastidiosa</i> <i>Journal of Bacteriology</i>,    185: 1018-1026.</div></li>
<li><div>8. Swofford DL (2000). <i>PAUP</i>*-<i>Phylogenetic Analysis Using Parsimony    </i>(*<i>and other methods</i>). Version 4. Sinauer Associates, Sunderland,    MA, USA.</div></li>
<li><div>9. Ochman H, Elwyn S &amp; Moran NA (1999). Calibrating bacterial evolution.    <i>Proceedings of the National Academy of Sciences, USA</i>, 96: 12638-12643.</div></li>
<li><div>10. Messer W (2002). The bacterial replication initiator DnaA. DnaA and oriC,    the bacterial mode to initiate DNA replication. <i>FEMS Microbiology Reviews</i>,    26: 355-374.</div></li>
<li><div>11. Lobry JR (1996). Asymmetric substitution patterns in the two DNA strands    of bacteria. <i>Molecular Biology and Evolution</i>, 13: 660-665.</div></li>
<li><div>12. Fukunaga M &amp; Mifuchi I (1989). Unique organization of <i>Leptospira    interrogans</i> rRNA genes. <i>Journal of Bacteriology</i>, 171: 5763-5767.</div></li>
<li><div>13. Paster BJ &amp; Dewhirst FE (2000). Phylogenetic foundation of spirochetes.    <i>Journal of Molecular Microbiology and Biotechnology</i>, 2: 341-344.</div></li>
<li><div>14. Postic D, Riquelme-Sertour N, Merien F, Perolat P &amp; Baranton G (2000).    Interest of partial 16S rDNA gene sequences to resolve heterogeneities between    <i>Leptospira</i> collections: application to <i>L. meyeri</i> <i>Research    in Microbiology</i>, 151: 333-341.</div></li>
<li><div>15. Heidelberg JF, Eisen JA, Nelson WC et al. (2000). DNA sequence of both    chromosomes of the cholera pathogen <i>Vibrio cholerae</i> <i>Nature</i>, 406:    477-483.</div></li>
<li><div>16. Salanoubat M, Genin S, Artiguenave F et al. (2002). Genome sequence of    the plant pathogen <i>Ralstonia solanacearum</i> <i>Nature</i>, 415: 497-502.</div></li>
<li><div>17. Bourhy P &amp; Saint Girons I (2000). Localization of the <i>Leptospira    interrogans</i> metF gene on the CII secondary chromosome. <i>FEMS Microbiology    Letters</i>, 191: 259-263.</div></li>
<li><div>18. Guegan R, Camadro JM, Saint Girons I &amp; Picardeau M (2003). <i>Leptospira</i>    spp. possess a complete haem biosynthetic pathway and are able to use exogenous    haem sources. <i>Molecular Microbiology</i>, 49: 745-754.</div></li>
<li><div>19. Rodionov DA, Vitreschak AG, Mironov AA &amp; Gelfand MS (2003). Comparative    genomics of the vitamin B12 metabolism and regulation in prokaryotes. <i>Journal    of Biological Chemistry</i>, 278: 41148-41159.</div></li>
<li><div>20. Spencer JB, Stolowich NJ, Roessner CA &amp; Scott AI (1993). The <i>Escherichia    coli</i> cysG gene encodes the multifunctional protein, siroheme synthase. <i>FEBS    Letters</i>, 335: 57-60.</div></li>
<li><div>21. Paulsen IT, Nguyen L, Sliwinski MK, Rabus R &amp; Saier Jr MH (2000).    Microbial genome analyses: comparative transport capabilities in eighteen prokaryotes.    <i>Journal of Molecular Biology</i>, 301: 75-100.</div></li>
<li><div>22. Meidanis J, Braga MD &amp; Verjovski-Almeida S (2002). Whole-genome analysis    of transporters in the plant pathogen <i>Xylella fastidiosa</i> <i>Microbiology    and Molecular Biology</i> <i>Reviews</i>, 66: 272-299.</div></li>
<li><div>23. Kammler M, Schon C &amp; Hantke K (1993). Characterization of the ferrous    iron uptake system of <i>Escherichia coli</i> <i>Journal of Bacteriology</i>,    175: 6212-6219.</div></li>
<li><div>24. Vujaskovic Z, Batinic-Haberle I, Rabbani ZN, Feng QF, Kang SK, Spasojevic    I, Samulski TV, Fridovich I, Dewhirst MW &amp; Anscher MS (2002). A small molecular    weight catalytic metalloporphyrin antioxidant with superoxide dismutase (SOD)    mimetic properties protects lungs from radiation-induced injury. <i>Free Radical    Biology and Medicine</i>, 33: 857-863.</div></li>
<li><div>25. Galperin MY, Nikolskaya AN &amp; Koonin EV (2001). Novel domains of the    prokaryotic two-component signal transduction systems. <i>FEMS Microbiology    Letters</i>, 203: 11-21.</div></li>
<li><div>26. Taylor BL &amp; Zhulin IB (1999). PAS domains: internal sensors of oxygen,    redox potential, and light. <i>Microbiology and Molecular Biology Reviews</i>,    63: 479-506.</div></li>
<li><div>27. Rickman L, Saldanha JW, Hunt DM, Hoar DN, Colston MJ, Millar JB &amp;    Buxton RS (2004). A two-component signal transduction system with a PAS domain-containing    sensor is required for virulence of <i>Mycobacterium tuberculosis</i> in mice.    <i>Biochemical and Biophysical Research Communications</i>, 314: 259-267.</div></li>
<li><div>28. Ausmees N, Mayer R, Weinhouse H, Volman G, Amikam D, Benziman M &amp;    Lindberg M (2001). Genetic data indicate that proteins containing the GGDEF    domain possess diguanylate cyclase activity. <i>FEMS Microbiology Letters</i>,    204: 163-167.</div></li>
<li><div>29. Anantharaman V &amp; Aravind L (2000). Cache - a signaling domain common    to animal Ca(2+)-channel subunits and a class of prokaryotic chemotaxis receptors.    <i>Trends in Biochemical</i> <i>Sciences</i>, 25: 535-537.</div></li>
<li><div>30. Ho YS, Burden LM &amp; Hurley JH (2000). Structure of the GAF domain,    a ubiquitous signaling motif and a new class of cyclic GMP receptor. <i>EMBO    Journal</i>, 19: 5288-5299.</div></li>
<li><div>31. Subramanian G, Koonin EV &amp; Aravind L (2000). Comparative genome analysis    of the pathogenic spirochetes <i>Borrelia burgdorferi</i> and <i>Treponema pallidum</i>    <i>Infection and Immunity</i>, 68: 1633-1648.</div></li>
<li><div>32. Bourret RB, Charon NW, Stock AM &amp; West AH (2002). Bright lights, abundant    operons-fluorescence and genomic technologies advance studies of bacterial locomotion    and signal transduction: review of the BLAST meeting, Cuernavaca, Mexico, 14    to 19 January 2001. <i>Journal of Bacteriology</i>, 184: 1-17.</div></li>
<li><div>33. Fraser CM, Casjens S, Huang WM et al. (1997). Genomic sequence of a Lyme    disease spirochaete, <i>Borrelia burgdorferi</i> <i>Nature</i>, 390: 580-586.</div></li>
<li><div>34. Fraser CM, Norris SJ, Weinstock GM et al. (1998). Complete genome sequence    of <i>Treponema pallidum</i>, the syphilis spirochete. <i>Science</i>, 281:    375-388.</div></li>
<li><div>35. Ge Y, Old IG, Saint Girons I &amp; Charon NW (1997). Molecular characterization    of a large <i>Borrelia burgdorferi</i> motility operon which is initiated by    a consensus sigma70 promoter. <i>Journal of Bacteriology</i>, 179: 2289-2299.</div></li>
<li><div>36. ter Huurne AA, Muir S, van Houten M, van der Zeijst BA, Gaastra W &amp;    Kusters JG (1994). Characterization of three putative <i>Serpulina hyodysenteriae</i>    hemolysins. <i>Microbial Pathogenesis</i>, 16: 269-282.</div></li>
<li><div>37. Squires C &amp; Squires CL (1992). The Clp proteins: proteolysis regulators    or molecular chaperones? <i>Journal of</i> <i>Bacteriology</i>, 174: 1081-1085.</div></li>
<li><div>38. Beem JE, Nesbitt WE &amp; Leung KP (1999). Cloning of <i>Prevotella intermedia</i>    loci demonstrating multiple hemolytic domains. <i>Oral Microbiology and Immunology</i>,    14: 143-152.</div></li>
<li><div>39. Lee SH, Kim KA, Park YG, Seong IW, Kim MJ &amp; Lee YJ (2000). Identification    and partial characterization of a novel hemolysin from <i>Leptospira interrogans</i>    serovar lai. <i>Gene</i>, 254: 19-28.</div></li>
<li><div>40. Shang ES, Exner MM, Summers TA, Martinich C, Champion CI, Hancock RE &amp;    Haake DA (1995). The rare outer membrane protein, OmpL1, of pathogenic <i>Leptospira</i>    species is a heat-modifiable porin. <i>Infection and Immunity</i>, 63: 3174-3181.</div></li>
<li><div>41. Voulhoux R, Bos MP, Geurtsen J, Mols M &amp; Tommassen J (2003). Role    of a highly conserved bacterial protein in outer membrane protein assembly.    <i>Science</i>, 299: 262-265.</div></li>
<li><div>42. Guffanti AA, Wei Y, Rood SV &amp; Krulwich TA (2002). An antiport mechanism    for a member of the cation diffusion facilitator family: divalent cations efflux    in exchange for K+ and H+. <i>Molecular Microbiology</i>, 45: 145-153.</div></li>
<li><div>43. Haake DA, Chao G, Zuerner RL, Barnett JK, Barnett D, Mazel M, Matsunaga    J, Levett PN &amp; Bolin CA (2000). The leptospiral major outer membrane protein    LipL32 is a lipoprotein expressed during mammalian infection. <i>Infection</i>    <i>and Immunity</i>, 68: 2276-2285.</div></li>
<li><div>44. Haake DA, Martinich C, Summers TA, Shang ES, Pruetz JD, McCoy AM, Mazel    MK &amp; Bolin CA (1998). Characterization of leptospiral outer membrane lipoprotein    LipL36: downregulation associated with late-log-phase growth and mammalian infection.    <i>Infection and Immunity</i>, 66: 1579-1587.</div></li>
<li><div>45. Shang ES, Summers TA &amp; Haake DA (1996). Molecular cloning and sequence    analysis of the gene encoding LipL41, a surface-exposed lipoprotein of pathogenic    <i>Leptospira</i> species. <i>Infection and Immunity</i>, 64: 2322-2330.</div></li>
<li><div>46. Haake DA &amp; Matsunaga J (2002). Characterization of the leptospiral    outer membrane and description of three novel leptospiral membrane proteins.    <i>Infection and Immunity</i>, 70: 4936-4945.</div></li>
<li><div>47. Haake DA (2000). Spirochaetal lipoproteins and pathogenesis. <i>Microbiology</i>,    146 (Part 7): 1491-1504.</div></li>
<li><div>48. Schuster B &amp; Sleytr UB (2000). S-layer-supported lipid membranes.    <i>Journal of Biotechnology</i>, 74: 233-254.</div></li>
<li><div>49. Matsunaga J, Young TA, Barnett JK, Barnett D, Bolin CA &amp; Haake DA    (2002). Novel 45-kilodalton leptospiral protein that is processed to a 31-kilodalton    growth-phase-regulated peripheral membrane protein. <i>Infection and Immunity</i>,    70: 323-334.</div></li>
<li><div>50. Cameron CE, Castro C, Lukehart SA &amp; Van Voorhis WC (1998). Function    and protective capacity of <i>Treponema</i> <i>pallidum</i> subsp. <i>pallidum</i>    glycerophosphodiester phosphodiesterase. <i>Infection and Immunity</i>, 66:    5763-5770.</div></li>
<li><div>51. De Mot R &amp; Vanderleyden J (1994). The C-terminal sequence conservation    between OmpA-related outer membrane proteins and MotB suggests a common function    in both gram-positive and gram-negative bacteria, possibly in the interaction    of these domains with peptidoglycan. <i>Molecular</i> <i>Microbiology</i>, 12:    333-334.</div></li>
<li><div>52. Kloda A &amp; Martinac B (2001). Structural and functional differences    between two homologous mechanosensitive channels of <i>Methanococcus jannaschii</i>    <i>EMBO Journal</i>, 20: 1888-1896.</div></li>
<li><div>53. de la Pena-Moctezuma A, Bulach DM, Kalambaheti T &amp; Adler B (1999).    Comparative analysis of the LPS biosynthetic loci of the genetic subtypes of    serovar Hardjo: <i>Leptospira interrogans</i> subtype <i>Hardjoprajitno</i>    and <i>Leptospira borgpetersenii</i> subtype <i>Hardjobovis</i> <i>FEMS</i>    <i>Microbiology Letters</i>, 177: 319-326.</div></li>
<li><div>54. Mitchison M, Bulach DM, Vinh T, Rajakumar K, Faine S &amp; Adler B (1997).    Identification and characterization of the dTDP-rhamnose biosynthesis and transfer    genes of the lipopolysaccharide-related rfb locus in <i>Leptospira</i> <i>interrogans    </i>serovar Copenhageni. <i>Journal of Bacteriology</i>, 179: 1262-1267.</div></li>
<li><div>55. Raetz CR &amp; Whitfield C (2002). Lipopolysaccharide endotoxins. <i>Annual    Review of Biochemistry</i>, 71: 635-700.</div></li>
<li><div>56. de Souza L &amp; Koury MC (1992). Isolation and biological activities    of endotoxin from <i>Leptospira interrogans</i> <i>Canadian Journal of Microbiology</i>,    38: 284-289.</div></li>
<li><div>57. Werts C, Tapping RI, Mathison JC et al. (2001). Leptospiral lipopolysaccharide    activates cells through a TLR2-dependent mechanism. <i>Nature Immunology</i>,    2: 346-352.</div></li>
<li><div>58. Boursaux-Eude C, Saint Girons I &amp; Zuerner R (1995). IS1500, an IS3-like    element from <i>Leptospira interrogans</i> <i>Microbiology</i>, 141 (Part 9):    2165-2173.</div></li>
<li><div>59. Zuerner RL (1994). Nucleotide sequence analysis of IS1533 from <i>Leptospira    borgpetersenii</i>: identification and expression of two IS-encoded proteins.    <i>Plasmid</i>, 31: 1-11.</div></li>
<li><div>60. Zuerner RL &amp; Huang WM (2002). Analysis of a <i>Leptospira interrogans</i>    locus containing DNA replication genes and a new IS, IS1502. <i>FEMS Microbiology    Letters</i>, 215: 175-182.</div></li>
</ul></div>
</div>
<div class="articleSection" data-anchor="Acknowledgments">
<h1 class="articleSectionTitle">Acknowledgments</h1>
<p>We are deeply indebted to Drs. I. Raw (Fundação Butantan, São Paulo, SP, Brazil) for use of laboratory facilities and valuable support. We thank Dr. Albert I. Ko (Fiocruz, Salvador, BA, Brazil) for the strain Fiocruz L1-130, the Agronomical and Environmental Genomes (AEG) Consortium of the network Organization for Nucleotide Sequencing and Analysis (ONSA) for the genome sequencing data, and Dr. L.C.C. Leite (Instituto Butantan) for a critical reading of this manuscript.</p>
</div>
<div class="articleSection" data-anchor="Correspondence and Footnotes">
<h1 class="articleSectionTitle">Correspondence and Footnotes</h1>
<div class="ref-list"><ul class="refList footnote">
<li><div>
<b>Address for correspondence:</b> A.L.T.O. Nascimento, Centro de Biotecnologia, Instituto Butantan, Av. Vital Brazil, 1500, 05503-900 São Paulo, SP, Brasil. Fax: +55-11-3726-1505. E-mail: <a href="mailto:tabet@butantan.gov.br">tabet@butantan.gov.br</a>
</div></li>
<li><div> Research supported by FAPESP and CNPq. Received February 20, 2004. Accepted March 8, 2004. </div></li>
</ul></div>
</div>
<div class="articleSection" data-anchor="Interviews">
<h1 class="articleSectionTitle">Interviews</h1>
<h2>Interview with Dr. José Fernando Perez</h2>
<p>
                <img style="max-width:100%" src="1414-431X-bjmbr-37-04-459-477-dr-peres.jpg">
                Dr. José Fernando Perez - Scientific Director, FAPESP
            </p>
<p>
                <video width="100%" controls="1"><source src="1414-431X-bjmbr-37-04-459-477-entrevista1.mp4" type="video/mp4"></source>
                    Your browser does not support the video element.
                </video>
            </p>
<h2>Interview with Prof. Lewis Joel Greene [BJMBR Editor]</h2>
<p>
                <img style="max-width:100%" src="1414-431X-bjmbr-37-04-459-477-dr-lewis.jpg">
                Prof. Lewis Joel Greene - BJMBR Editor
            </p>
<p>
                <video width="100%" controls="1"><source src="1414-431X-bjmbr-37-04-459-477-entrevista2.mp4" type="video/mp4"></source>
                    Your browser does not support the video element.
                </video>
            </p>
<h2>Interview with Dra. Ana Lucia Nascimento [First author, Butantã Institute]</h2>
<p>
                <img style="max-width:100%" src="1414-431X-bjmbr-37-04-459-477-dr-lucia.jpg">
                Dra. Ana Lucia Nascimento - First author - Butantã Institute
            </p>
<p>
                <video width="100%" controls="1"><source src="1414-431X-bjmbr-37-04-459-477-entrevista3.mp4" type="video/mp4"></source>
                    Your browser does not support the video element.
                </video>
            </p>
</div>
<div class="articleSection" data-anchor="Publication Dates">
<h1 class="articleSectionTitle">Publication Dates</h1>
<div class="row"><div class="col-md-12 col-sm-12"><ul class="articleTimeline">
<li>
<strong>Publication in this collection</strong><br>22 Apr 2004</li>
<li>
<strong>Date of issue</strong><br>Apr 2004</li>
</ul></div></div>
</div>
<div class="articleSection" data-anchor="History">
<h1 class="articleSectionTitle">History</h1>
<div class="row"><div class="col-md-12 col-sm-12"><ul class="articleTimeline">
<li>
<strong>Accepted</strong><br>08 Mar 2004</li>
<li>
<strong>Received</strong><br>20 Feb 2004</li>
</ul></div></div>
</div>
<section class="documentLicense"><div class="container-license"><div class="row">
<div class="col-sm-3 col-md-2"><a href="http://creativecommons.org/licenses/by/4.0/" target="_blank" title=""><img src="https://licensebuttons.net/l/by/4.0//88x31.png" alt="Creative Common - by 4.0 "></a></div>
<div class="col-sm-9 col-md-10"><a href="http://creativecommons.org/licenses/by/4.0/" target="_blank" title="">This work is licensed under a Creative Commons Attribution 4.0 International License.</a></div>
</div></div></section></article>
</div>
</div></div></section><div class="modal fade ModalDefault ModalTutors" id="ModalTutors" tabindex="-1" role="dialog" aria-hidden="true"><div class="modal-dialog"><div class="modal-content">
<div class="modal-header">
<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">×</span><span class="sr-only">Close</span></button><h4 class="modal-title">Authorship</h4>
</div>
<div class="modal-body"><div class="info">
<div class="tutors">
<strong> A.L.T.O. Nascimento </strong><span class="ref footnote"><sup class="xref"></sup><span class="refCtt closed"><span class="refCttPadding">Address for correspondence: A.L.T.O. Nascimento, Centro de Biotecnologia, Instituto Butantan, Av. Vital Brazil, 1500, 05503-900 São Paulo, SP, Brasil. Fax: +55-11-3726-1505. E-mail: tabet@butantan.gov.br</span></span></span><br><div>
<span data-aff-display="aff01">Instituto Butantan, Centro de Biotecnologia, São Paulo, SP, Brasil</span><span data-aff-orgname="aff01" hidden="">Instituto Butantan</span><span data-aff-country="aff01" hidden="">Brasil</span><span data-aff-country-code="aff01" hidden="" country="BR"></span><span data-aff-location="aff01" hidden="">São Paulo, SP, Brasil</span><span data-aff-full="aff01" hidden="">Instituto Butantan, Centro de Biotecnologia, São Paulo, SP, Brasil</span><a href="" class="scimago-link" data-scimago-link="aff01"></a>
</div>Address for correspondence: A.L.T.O. Nascimento, Centro de Biotecnologia, Instituto Butantan, Av. Vital Brazil, 1500, 05503-900 São Paulo, SP, Brasil. Fax: +55-11-3726-1505. E-mail: tabet@butantan.gov.br<div class="clearfix"></div>
</div>
<div class="tutors">
<strong> S. Verjovski-Almeida </strong><br><div>
<span data-aff-display="aff02">Universidade de São Paulo, Instituto de Química, São Paulo, Brazil</span><span data-aff-orgname="aff02" hidden="">Universidade de São Paulo</span><span data-aff-country="aff02" hidden="">Brazil</span><span data-aff-country-code="aff02" hidden="" country="BR"></span><span data-aff-location="aff02" hidden="">São Paulo, Brazil</span><span data-aff-full="aff02" hidden="">Universidade de São Paulo, Instituto de Química, São Paulo, Brazil</span><a href="" class="scimago-link" data-scimago-link="aff02"></a>
</div>
<div class="clearfix"></div>
</div>
<div class="tutors">
<strong> M.A. Van Sluys </strong><br><div>
<span data-aff-display="aff03">Universidade de São Paulo, Instituto de Biociências, São Paulo, Brazil</span><span data-aff-orgname="aff03" hidden="">Universidade de São Paulo</span><span data-aff-country="aff03" hidden="">Brazil</span><span data-aff-country-code="aff03" hidden="" country="BR"></span><span data-aff-location="aff03" hidden="">São Paulo, Brazil</span><span data-aff-full="aff03" hidden="">Universidade de São Paulo, Instituto de Biociências, São Paulo, Brazil</span><a href="" class="scimago-link" data-scimago-link="aff03"></a>
</div>
<div class="clearfix"></div>
</div>
<div class="tutors">
<strong> C.B. Monteiro-Vitorello </strong><br><div>
<span data-aff-display="aff05">Universidade de São Paulo, Escola Superior de Agricultura Luiz de Queiroz, Piracicaba, São Paulo, Brazil</span><span data-aff-orgname="aff05" hidden="">Universidade de São Paulo</span><span data-aff-country="aff05" hidden="">Brazil</span><span data-aff-country-code="aff05" hidden="" country="BR"></span><span data-aff-location="aff05" hidden="">Piracicaba, São Paulo, Brazil</span><span data-aff-full="aff05" hidden="">Universidade de São Paulo, Escola Superior de Agricultura Luiz de Queiroz, Piracicaba, São Paulo, Brazil</span><a href="" class="scimago-link" data-scimago-link="aff05"></a>
</div>
<div class="clearfix"></div>
</div>
<div class="tutors">
<strong> L.E.A. Camargo </strong><br><div>
<span data-aff-display="aff05">Universidade de São Paulo, Escola Superior de Agricultura Luiz de Queiroz, Piracicaba, São Paulo, Brazil</span><span data-aff-orgname="aff05" hidden="">Universidade de São Paulo</span><span data-aff-country="aff05" hidden="">Brazil</span><span data-aff-country-code="aff05" hidden="" country="BR"></span><span data-aff-location="aff05" hidden="">Piracicaba, São Paulo, Brazil</span><span data-aff-full="aff05" hidden="">Universidade de São Paulo, Escola Superior de Agricultura Luiz de Queiroz, Piracicaba, São Paulo, Brazil</span><a href="" class="scimago-link" data-scimago-link="aff05"></a>
</div>
<div class="clearfix"></div>
</div>
<div class="tutors">
<strong> L.A. Digiampietri </strong><br><div>
<span data-aff-display="aff06">Universidade Estadual de Campinas, Instituto de Computação, Laboratório de Bioinformática, Campinas, São Paulo, Brazil</span><span data-aff-orgname="aff06" hidden="">Universidade Estadual de Campinas</span><span data-aff-country="aff06" hidden="">Brazil</span><span data-aff-country-code="aff06" hidden="" country="BR"></span><span data-aff-location="aff06" hidden="">Campinas, São Paulo, Brazil</span><span data-aff-full="aff06" hidden="">Universidade Estadual de Campinas, Instituto de Computação, Laboratório de Bioinformática, Campinas, São Paulo, Brazil</span><a href="" class="scimago-link" data-scimago-link="aff06"></a>
</div>
<div class="clearfix"></div>
</div>
<div class="tutors">
<strong> R.A. Harstkeerl </strong><br><div>
<span data-aff-display="aff07">KIT Biomedical Research, Koninklijk Instituut voor de Tropen/Royal Tropical Institute (KIT), Amsterdam, The Netherlands</span><span data-aff-orgname="aff07" hidden="">KIT Biomedical Research</span><span data-aff-country="aff07" hidden="">The Netherlands</span><span data-aff-country-code="aff07" hidden="" country=""></span><span data-aff-location="aff07" hidden="">Amsterdam, The Netherlands</span><span data-aff-full="aff07" hidden="">KIT Biomedical Research, Koninklijk Instituut voor de Tropen/Royal Tropical Institute (KIT), Amsterdam, The Netherlands</span><a href="" class="scimago-link" data-scimago-link="aff07"></a>
</div>
<div class="clearfix"></div>
</div>
<div class="tutors">
<strong> P.L. Ho </strong><br><div>
<span data-aff-display="aff01">Instituto Butantan, Centro de Biotecnologia, São Paulo, SP, Brasil</span><span data-aff-orgname="aff01" hidden="">Instituto Butantan</span><span data-aff-country="aff01" hidden="">Brasil</span><span data-aff-country-code="aff01" hidden="" country="BR"></span><span data-aff-location="aff01" hidden="">São Paulo, SP, Brasil</span><span data-aff-full="aff01" hidden="">Instituto Butantan, Centro de Biotecnologia, São Paulo, SP, Brasil</span><a href="" class="scimago-link" data-scimago-link="aff01"></a>
</div>
<div class="clearfix"></div>
</div>
<div class="tutors">
<strong> M.V. Marques </strong><br><div>
<span data-aff-display="aff04">Universidade de São Paulo, Instituto de Ciências Biomédicas, São Paulo, São Paulo, Brazil</span><span data-aff-orgname="aff04" hidden="">Universidade de São Paulo</span><span data-aff-country="aff04" hidden="">Brazil</span><span data-aff-country-code="aff04" hidden="" country="BR"></span><span data-aff-location="aff04" hidden="">São Paulo, São Paulo, Brazil</span><span data-aff-full="aff04" hidden="">Universidade de São Paulo, Instituto de Ciências Biomédicas, São Paulo, São Paulo, Brazil</span><a href="" class="scimago-link" data-scimago-link="aff04"></a>
</div>
<div class="clearfix"></div>
</div>
<div class="tutors">
<strong> M.C. Oliveira </strong><br><div>
<span data-aff-display="aff03">Universidade de São Paulo, Instituto de Biociências, São Paulo, Brazil</span><span data-aff-orgname="aff03" hidden="">Universidade de São Paulo</span><span data-aff-country="aff03" hidden="">Brazil</span><span data-aff-country-code="aff03" hidden="" country="BR"></span><span data-aff-location="aff03" hidden="">São Paulo, Brazil</span><span data-aff-full="aff03" hidden="">Universidade de São Paulo, Instituto de Biociências, São Paulo, Brazil</span><a href="" class="scimago-link" data-scimago-link="aff03"></a>
</div>
<div class="clearfix"></div>
</div>
<div class="tutors">
<strong> J.C. Setubal </strong><br><div>
<span data-aff-display="aff06">Universidade Estadual de Campinas, Instituto de Computação, Laboratório de Bioinformática, Campinas, São Paulo, Brazil</span><span data-aff-orgname="aff06" hidden="">Universidade Estadual de Campinas</span><span data-aff-country="aff06" hidden="">Brazil</span><span data-aff-country-code="aff06" hidden="" country="BR"></span><span data-aff-location="aff06" hidden="">Campinas, São Paulo, Brazil</span><span data-aff-full="aff06" hidden="">Universidade Estadual de Campinas, Instituto de Computação, Laboratório de Bioinformática, Campinas, São Paulo, Brazil</span><a href="" class="scimago-link" data-scimago-link="aff06"></a>
</div>
<div>
<span data-aff-display="aff10">Virginia Polytechnic Institute and State University Bioinformatics 1, Virginia Bioinformatics Institute and Department of Computer Science, Blacksburg, VA, USA</span><span data-aff-orgname="aff10" hidden="">Virginia Polytechnic Institute and State University Bioinformatics 1</span><span data-aff-country="aff10" hidden="">USA</span><span data-aff-country-code="aff10" hidden="" country="US"></span><span data-aff-location="aff10" hidden="">Blacksburg, VA, USA</span><span data-aff-full="aff10" hidden="">Virginia Polytechnic Institute and State University Bioinformatics 1, Virginia Bioinformatics Institute and Department of Computer Science, Blacksburg, VA, USA</span><a href="" class="scimago-link" data-scimago-link="aff10"></a>
</div>
<div class="clearfix"></div>
</div>
<div class="tutors">
<strong> D.A. Haake </strong><br><div>
<span data-aff-display="aff08">The David Geffen School of Medicine at University of California, Los Angeles, CA, USA</span><span data-aff-orgname="aff08" hidden="">The David Geffen School of Medicine at University of California</span><span data-aff-country="aff08" hidden="">USA</span><span data-aff-country-code="aff08" hidden="" country="US"></span><span data-aff-location="aff08" hidden="">Los Angeles, CA, USA</span><span data-aff-full="aff08" hidden="">The David Geffen School of Medicine at University of California, Los Angeles, CA, USA</span><a href="" class="scimago-link" data-scimago-link="aff08"></a>
</div>
<div>
<span data-aff-display="aff09">Veterans Affairs Greater Los Angeles Healthcare System, Division of Infectious Diseases, Los Angeles, CA, USA</span><span data-aff-orgname="aff09" hidden="">Veterans Affairs Greater Los Angeles Healthcare System</span><span data-aff-country="aff09" hidden="">USA</span><span data-aff-country-code="aff09" hidden="" country="US"></span><span data-aff-location="aff09" hidden="">Los Angeles, CA, USA</span><span data-aff-full="aff09" hidden="">Veterans Affairs Greater Los Angeles Healthcare System, Division of Infectious Diseases, Los Angeles, CA, USA</span><a href="" class="scimago-link" data-scimago-link="aff09"></a>
</div>
<div class="clearfix"></div>
</div>
<div class="tutors">
<strong> E.A.L. Martins </strong><br><div>
<span data-aff-display="aff01">Instituto Butantan, Centro de Biotecnologia, São Paulo, SP, Brasil</span><span data-aff-orgname="aff01" hidden="">Instituto Butantan</span><span data-aff-country="aff01" hidden="">Brasil</span><span data-aff-country-code="aff01" hidden="" country="BR"></span><span data-aff-location="aff01" hidden="">São Paulo, SP, Brasil</span><span data-aff-full="aff01" hidden="">Instituto Butantan, Centro de Biotecnologia, São Paulo, SP, Brasil</span><a href="" class="scimago-link" data-scimago-link="aff01"></a>
</div>
<div class="clearfix"></div>
</div>
</div></div>
</div></div></div>
<div class="modal fade ModalDefault ModalTutors" id="ModalScimago" tabindex="-1" role="dialog" aria-hidden="true"><div class="modal-dialog"><div class="modal-content">
<div class="modal-header">
<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">×</span><span class="sr-only">Close</span></button><h4 class="modal-title">
                            SCIMAGO INSTITUTIONS RANKINGS
                        </h4>
</div>
<div class="modal-body"><div class="info">
<div>
<span data-aff-display="aff05">Universidade de São Paulo, Escola Superior de Agricultura Luiz de Queiroz, Piracicaba, São Paulo, Brazil</span><span data-aff-orgname="aff05" hidden="">Universidade de São Paulo</span><span data-aff-country="aff05" hidden="">Brazil</span><span data-aff-country-code="aff05" hidden="" country="BR"></span><span data-aff-location="aff05" hidden="">Piracicaba, São Paulo, Brazil</span><span data-aff-full="aff05" hidden="">Universidade de São Paulo, Escola Superior de Agricultura Luiz de Queiroz, Piracicaba, São Paulo, Brazil</span><a href="" class="scimago-link" data-scimago-link="aff05"></a><br>
</div>
<div>
<span data-aff-display="aff10">Virginia Polytechnic Institute and State University Bioinformatics 1, Virginia Bioinformatics Institute and Department of Computer Science, Blacksburg, VA, USA</span><span data-aff-orgname="aff10" hidden="">Virginia Polytechnic Institute and State University Bioinformatics 1</span><span data-aff-country="aff10" hidden="">USA</span><span data-aff-country-code="aff10" hidden="" country="US"></span><span data-aff-location="aff10" hidden="">Blacksburg, VA, USA</span><span data-aff-full="aff10" hidden="">Virginia Polytechnic Institute and State University Bioinformatics 1, Virginia Bioinformatics Institute and Department of Computer Science, Blacksburg, VA, USA</span><a href="" class="scimago-link" data-scimago-link="aff10"></a><br>
</div>
<div>
<span data-aff-display="aff06">Universidade Estadual de Campinas, Instituto de Computação, Laboratório de Bioinformática, Campinas, São Paulo, Brazil</span><span data-aff-orgname="aff06" hidden="">Universidade Estadual de Campinas</span><span data-aff-country="aff06" hidden="">Brazil</span><span data-aff-country-code="aff06" hidden="" country="BR"></span><span data-aff-location="aff06" hidden="">Campinas, São Paulo, Brazil</span><span data-aff-full="aff06" hidden="">Universidade Estadual de Campinas, Instituto de Computação, Laboratório de Bioinformática, Campinas, São Paulo, Brazil</span><a href="" class="scimago-link" data-scimago-link="aff06"></a><br>
</div>
<div>
<span data-aff-display="aff09">Veterans Affairs Greater Los Angeles Healthcare System, Division of Infectious Diseases, Los Angeles, CA, USA</span><span data-aff-orgname="aff09" hidden="">Veterans Affairs Greater Los Angeles Healthcare System</span><span data-aff-country="aff09" hidden="">USA</span><span data-aff-country-code="aff09" hidden="" country="US"></span><span data-aff-location="aff09" hidden="">Los Angeles, CA, USA</span><span data-aff-full="aff09" hidden="">Veterans Affairs Greater Los Angeles Healthcare System, Division of Infectious Diseases, Los Angeles, CA, USA</span><a href="" class="scimago-link" data-scimago-link="aff09"></a><br>
</div>
<div>
<span data-aff-display="aff08">The David Geffen School of Medicine at University of California, Los Angeles, CA, USA</span><span data-aff-orgname="aff08" hidden="">The David Geffen School of Medicine at University of California</span><span data-aff-country="aff08" hidden="">USA</span><span data-aff-country-code="aff08" hidden="" country="US"></span><span data-aff-location="aff08" hidden="">Los Angeles, CA, USA</span><span data-aff-full="aff08" hidden="">The David Geffen School of Medicine at University of California, Los Angeles, CA, USA</span><a href="" class="scimago-link" data-scimago-link="aff08"></a><br>
</div>
<div>
<span data-aff-display="aff01">Instituto Butantan, Centro de Biotecnologia, São Paulo, SP, Brasil</span><span data-aff-orgname="aff01" hidden="">Instituto Butantan</span><span data-aff-country="aff01" hidden="">Brasil</span><span data-aff-country-code="aff01" hidden="" country="BR"></span><span data-aff-location="aff01" hidden="">São Paulo, SP, Brasil</span><span data-aff-full="aff01" hidden="">Instituto Butantan, Centro de Biotecnologia, São Paulo, SP, Brasil</span><a href="" class="scimago-link" data-scimago-link="aff01"></a><br>
</div>
<div>
<span data-aff-display="aff02">Universidade de São Paulo, Instituto de Química, São Paulo, Brazil</span><span data-aff-orgname="aff02" hidden="">Universidade de São Paulo</span><span data-aff-country="aff02" hidden="">Brazil</span><span data-aff-country-code="aff02" hidden="" country="BR"></span><span data-aff-location="aff02" hidden="">São Paulo, Brazil</span><span data-aff-full="aff02" hidden="">Universidade de São Paulo, Instituto de Química, São Paulo, Brazil</span><a href="" class="scimago-link" data-scimago-link="aff02"></a><br>
</div>
<div>
<span data-aff-display="aff04">Universidade de São Paulo, Instituto de Ciências Biomédicas, São Paulo, São Paulo, Brazil</span><span data-aff-orgname="aff04" hidden="">Universidade de São Paulo</span><span data-aff-country="aff04" hidden="">Brazil</span><span data-aff-country-code="aff04" hidden="" country="BR"></span><span data-aff-location="aff04" hidden="">São Paulo, São Paulo, Brazil</span><span data-aff-full="aff04" hidden="">Universidade de São Paulo, Instituto de Ciências Biomédicas, São Paulo, São Paulo, Brazil</span><a href="" class="scimago-link" data-scimago-link="aff04"></a><br>
</div>
<div>
<span data-aff-display="aff07">KIT Biomedical Research, Koninklijk Instituut voor de Tropen/Royal Tropical Institute (KIT), Amsterdam, The Netherlands</span><span data-aff-orgname="aff07" hidden="">KIT Biomedical Research</span><span data-aff-country="aff07" hidden="">The Netherlands</span><span data-aff-country-code="aff07" hidden="" country=""></span><span data-aff-location="aff07" hidden="">Amsterdam, The Netherlands</span><span data-aff-full="aff07" hidden="">KIT Biomedical Research, Koninklijk Instituut voor de Tropen/Royal Tropical Institute (KIT), Amsterdam, The Netherlands</span><a href="" class="scimago-link" data-scimago-link="aff07"></a><br>
</div>
<div>
<span data-aff-display="aff03">Universidade de São Paulo, Instituto de Biociências, São Paulo, Brazil</span><span data-aff-orgname="aff03" hidden="">Universidade de São Paulo</span><span data-aff-country="aff03" hidden="">Brazil</span><span data-aff-country-code="aff03" hidden="" country="BR"></span><span data-aff-location="aff03" hidden="">São Paulo, Brazil</span><span data-aff-full="aff03" hidden="">Universidade de São Paulo, Instituto de Biociências, São Paulo, Brazil</span><a href="" class="scimago-link" data-scimago-link="aff03"></a><br>
</div>
</div></div>
</div></div></div>
<div class="modal fade ModalDefault" id="ModalTablesFigures" tabindex="-1" role="dialog" aria-hidden="true"><div class="modal-dialog"><div class="modal-content">
<div class="modal-header">
<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">×</span><span class="sr-only">Close</span></button><h4 class="modal-title">Figures | Tables</h4>
</div>
<div class="modal-body">
<ul class="nav nav-tabs md-tabs" role="tablist">
<li role="presentation" class="col-md-4 col-sm-4 active"><a href="#figures" aria-controls="figures" role="tab" data-toggle="tab">Figures
                    (7)
                </a></li>
<li role="presentation" class="col-md-4 col-sm-4 "><a href="#tables" aria-controls="tables" role="tab" data-toggle="tab">Tables
                    (2)
                </a></li>
</ul>
<div class="clearfix"></div>
<div class="tab-content">
<div role="tabpanel" class="tab-pane active" id="figures">
<div class="row fig">
<div class="col-md-4"><a data-toggle="modal" data-target="#ModalFigfig1"><div class="thumbOff">
                        Thumbnail
                        <div class="zoom"><span class="sci-ico-zoom"></span></div>
</div></a></div>
<div class="col-md-8">
<strong>Figure 1.</strong><br> Circular maps of the two chromosomes in <i>Leptospira interrogans.</i> Circular representation of the <i>L. interrogans</i> serovar Copenhageni genome. The two large and small chromosomes are presented. Circles 1 and 2 (from the outside in), all predicted protein-coding regions (forward and reverse strand, respectively) color-coded by category role; circle 3: G + C content. Numbers on the outer circle are base pairs. rRNA genes, <i>rrl</i>, <i>rrs</i> and <i>rrf</i> are 23S, 16S and 5S, respectively. Note: the two replicons are not drawn to the same scale. Chromosome II (CII) is 12 times smaller than chromosome I (CI). </div>
</div>
<div class="row fig">
<div class="col-md-4"><a data-toggle="modal" data-target="#ModalFigfig2"><div class="thumbOff">
                        Thumbnail
                        <div class="zoom"><span class="sci-ico-zoom"></span></div>
</div></a></div>
<div class="col-md-8">
<strong>Figure 2.</strong><br> Phylogenetic analysis of <i>Leptospira interrogans</i>. Consensus phylogenetic distance tree constructed using 16S rDNA sequences. Numbers on the branches are bootstrap values (2000 replicates) for distance (top), parsimony (below the branch) and maximum likelihood (10 replicates, below the branch, italics). GenBank accession numbers are in parenteses. <i>Leptonema</i> is used as outgroup. <i>A</i>, Based on a matrix with 34 taxa and 1255 positions; <i>B</i>, based on a matrix with 24 taxa and 1375 positions. </div>
</div>
<div class="row fig">
<div class="col-md-4"><a data-toggle="modal" data-target="#ModalFigfig3"><div class="thumbOff">
                        Thumbnail
                        <div class="zoom"><span class="sci-ico-zoom"></span></div>
</div></a></div>
<div class="col-md-8">
<strong>Figure 3.</strong><br> The cobalamin biosynthetic pathway of <i>Leptospira interrogans</i>. Genetic organization of the cobalamin biosynthetic locus in the two chromosomes (CI and CII) of <i>L. interrogans</i> serovar Copenhageni. </div>
</div>
<div class="row fig">
<div class="col-md-4"><a data-toggle="modal" data-target="#ModalFigfig4"><div class="thumbOff">
                        Thumbnail
                        <div class="zoom"><span class="sci-ico-zoom"></span></div>
</div></a></div>
<div class="col-md-8">
<strong>Figure 4.</strong><br> Major transporters of <i>Leptospira interrogans</i>. Major primary transporters are driven by ATP hydrolysis and include ABC-transporters, P-type ion transporters and the ATP-synthase. All other transporters indicated are the major secondary transporters, which are driven by the transmembrane electrochemical potential. RND = resistance-nodulation-cell division superfamily. </div>
</div>
<div class="row fig">
<div class="col-md-4"><a data-toggle="modal" data-target="#ModalFigfig5"><div class="thumbOff">
                        Thumbnail
                        <div class="zoom"><span class="sci-ico-zoom"></span></div>
</div></a></div>
<div class="col-md-8">
<strong>Figure 5.</strong><br> Domain architecture of putative signal transduction proteins from <i>Leptospira interrogans.</i> The domain organization of histidine kinases, response regulators, hybrid histidine kinase/response regulators and other peptides probably associated with signal transduction is shown schematically. The number of peptides containing each domain organization is shown on the right, and the genes are found distributed over the large or small replicons, except for the seven peptides containing the PAS/GGDEF arrangement, which are encoded by genes located in tandem in the large replicon. AAA = ATPase; Cache = small ligand binding domain; EAL = phosphodiesterase; GAF = cGMP-binding domain; GGDEF = diguanylate cyclase; HK = histidine kinase domain; HTH = helix-turn-helix; PAS = sensor domain (may include a PAS/PAC arrangement); PP = phosphatase; RR = response regulator domain; SP = signal peptide; TM = transmembrane region. </div>
</div>
<div class="row fig">
<div class="col-md-4"><a data-toggle="modal" data-target="#ModalFigfig6"><div class="thumbOff">
                        Thumbnail
                        <div class="zoom"><span class="sci-ico-zoom"></span></div>
</div></a></div>
<div class="col-md-8">
<strong>Figure 6.</strong><br> Model of leptospiral membrane architecture. Leptospires have two membranes, an outer membrane (OM) and a cytoplasmic or inner membrane (IM). As in Gram-positive bacteria, the peptidoglycan (PG) cell wall is closely associated with the IM. The leptospiral surface is dominated by lipopolysaccharide (LPS) carbohydrate side chains. Subsurface proteins include the cytoplasmic protein, GroEL, and the periplasmic endoflagella (EF). The IM contains lipoproteins such as LipL31 and transmembrane proteins such as signal peptidase (SP) and ImpL63. The OM contains lipoproteins including LipL41 and LipL36, and transmembrane proteins including the porin, OmpL1. Genomic sequence analysis reveals several novel types of outer membrane proteins (OMPs), including TonB-dependent OMPs involved in nutrient acquisition. BtuB is an example of a TonB-dependent OMP involved in uptake of vitamin B<sub>12</sub>. The type I efflux system is represented by TolC, which forms a complex with ATP-binding cassette (ABC) transporters to export hemolysins and other cytoplasmic components. The leptospiral genome also contains genes involved in a three-component efflux system consisting of an outer membrane factor (OMF), membrane fusion protein (MFP), and an inner membrane transporter, in this case CzcA, which is involved in heavy metal detoxification. </div>
</div>
<div class="row fig">
<div class="col-md-4"><a data-toggle="modal" data-target="#ModalFigfig7"><div class="thumbOff">
                        Thumbnail
                        <div class="zoom"><span class="sci-ico-zoom"></span></div>
</div></a></div>
<div class="col-md-8">
<strong>Figure 7.</strong><br> Lipopolysaccharide region of <i>Leptospira interrogans.</i> The 119-kb region of <i>L. interrogans</i> serovar Copenhageni genome with 108 predicted coding sequences transcribed in the same orientation. Orange arrows indicate predicted genes encoding polysaccharide biosynthesis-related proteins. Shaded regions indicate the homologous genes from <i>L. borgpetersenii</i> serovar Hardjobovis (53). The region containing the predicted coding sequences, indicated by the letters <i>F, E, A, B, D, C</i>, was first indicated as the <i>rfb</i> genes related to rhamnose biosynthesis in <i>L. interrogans</i> Copenhageni (54). </div>
</div>
</div>
<div role="tabpanel" class="tab-pane " id="tables">
<div class="row table">
<div class="col-md-4"><a data-toggle="modal" data-target="#ModalTablet01"><div class="thumbOff">
                        Thumbnail
                        <div class="zoom"><span class="sci-ico-zoom"></span></div>
</div></a></div>
<div class="col-md-8"></div>
</div>
<div class="row table">
<div class="col-md-4"><a data-toggle="modal" data-target="#ModalTablet02"><div class="thumbOff">
                        Thumbnail
                        <div class="zoom"><span class="sci-ico-zoom"></span></div>
</div></a></div>
<div class="col-md-8"></div>
</div>
</div>
</div>
</div>
</div></div></div>
<div class="modal fade ModalFigs" id="ModalFigfig1" tabindex="-1" role="dialog" aria-hidden="true"><div class="modal-dialog modal-lg"><div class="modal-content">
<div class="modal-header">
<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">×</span><span class="sr-only">Close</span></button><a class="link-newWindow showTooltip" target="_blank" data-placement="left" title="Open new window" href="1414-431X-bjmbr-37-04-459-477-g5500i01.jpg"><span class="sci-ico-newWindow"></span></a><h4 class="modal-title">
<span class="sci-ico-fileFigure"></span>Figure 1.    Circular maps of the two chromosomes in <i>Leptospira interrogans.</i> Circular representation of the <i>L. interrogans</i> serovar Copenhageni genome. The two large and small chromosomes are presented. Circles 1 and 2 (from the outside in), all predicted protein-coding regions (forward and reverse strand, respectively) color-coded by category role; circle 3: G + C content. Numbers on the outer circle are base pairs. rRNA genes, <i>rrl</i>, <i>rrs</i> and <i>rrf</i> are 23S, 16S and 5S, respectively. Note: the two replicons are not drawn to the same scale. Chromosome II (CII) is 12 times smaller than chromosome I (CI). <br>
</h4>
</div>
<div class="modal-body"><img style="max-width:100%" src="1414-431X-bjmbr-37-04-459-477-g5500i01.jpg"></div>
</div></div></div>
<div class="modal fade ModalFigs" id="ModalFigfig2" tabindex="-1" role="dialog" aria-hidden="true"><div class="modal-dialog modal-lg"><div class="modal-content">
<div class="modal-header">
<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">×</span><span class="sr-only">Close</span></button><a class="link-newWindow showTooltip" target="_blank" data-placement="left" title="Open new window" href="1414-431X-bjmbr-37-04-459-477-g5500i02.jpg"><span class="sci-ico-newWindow"></span></a><h4 class="modal-title">
<span class="sci-ico-fileFigure"></span>Figure 2.    Phylogenetic analysis of <i>Leptospira interrogans</i>. Consensus phylogenetic distance tree constructed using 16S rDNA sequences. Numbers on the branches are bootstrap values (2000 replicates) for distance (top), parsimony (below the branch) and maximum likelihood (10 replicates, below the branch, italics). GenBank accession numbers are in parenteses. <i>Leptonema</i> is used as outgroup. <i>A</i>, Based on a matrix with 34 taxa and 1255 positions; <i>B</i>, based on a matrix with 24 taxa and 1375 positions. <br>
</h4>
</div>
<div class="modal-body"><img style="max-width:100%" src="1414-431X-bjmbr-37-04-459-477-g5500i02.jpg"></div>
</div></div></div>
<div class="modal fade ModalFigs" id="ModalFigfig3" tabindex="-1" role="dialog" aria-hidden="true"><div class="modal-dialog modal-lg"><div class="modal-content">
<div class="modal-header">
<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">×</span><span class="sr-only">Close</span></button><a class="link-newWindow showTooltip" target="_blank" data-placement="left" title="Open new window" href="1414-431X-bjmbr-37-04-459-477-g5500i03.jpg"><span class="sci-ico-newWindow"></span></a><h4 class="modal-title">
<span class="sci-ico-fileFigure"></span>Figure 3.    The cobalamin biosynthetic pathway of <i>Leptospira interrogans</i>. Genetic organization of the cobalamin biosynthetic locus in the two chromosomes (CI and CII) of <i>L. interrogans</i> serovar Copenhageni. <br>
</h4>
</div>
<div class="modal-body"><img style="max-width:100%" src="1414-431X-bjmbr-37-04-459-477-g5500i03.jpg"></div>
</div></div></div>
<div class="modal fade ModalFigs" id="ModalFigfig4" tabindex="-1" role="dialog" aria-hidden="true"><div class="modal-dialog modal-lg"><div class="modal-content">
<div class="modal-header">
<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">×</span><span class="sr-only">Close</span></button><a class="link-newWindow showTooltip" target="_blank" data-placement="left" title="Open new window" href="1414-431X-bjmbr-37-04-459-477-g5500i04.jpg"><span class="sci-ico-newWindow"></span></a><h4 class="modal-title">
<span class="sci-ico-fileFigure"></span>Figure 4.    Major transporters of <i>Leptospira interrogans</i>. Major primary transporters are driven by ATP hydrolysis and include ABC-transporters, P-type ion transporters and the ATP-synthase. All other transporters indicated are the major secondary transporters, which are driven by the transmembrane electrochemical potential. RND = resistance-nodulation-cell division superfamily. <br>
</h4>
</div>
<div class="modal-body"><img style="max-width:100%" src="1414-431X-bjmbr-37-04-459-477-g5500i04.jpg"></div>
</div></div></div>
<div class="modal fade ModalFigs" id="ModalFigfig5" tabindex="-1" role="dialog" aria-hidden="true"><div class="modal-dialog modal-lg"><div class="modal-content">
<div class="modal-header">
<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">×</span><span class="sr-only">Close</span></button><a class="link-newWindow showTooltip" target="_blank" data-placement="left" title="Open new window" href="1414-431X-bjmbr-37-04-459-477-g5500i05.jpg"><span class="sci-ico-newWindow"></span></a><h4 class="modal-title">
<span class="sci-ico-fileFigure"></span>Figure 5.    Domain architecture of putative signal transduction proteins from <i>Leptospira interrogans.</i> The domain organization of histidine kinases, response regulators, hybrid histidine kinase/response regulators and other peptides probably associated with signal transduction is shown schematically. The number of peptides containing each domain organization is shown on the right, and the genes are found distributed over the large or small replicons, except for the seven peptides containing the PAS/GGDEF arrangement, which are encoded by genes located in tandem in the large replicon. AAA = ATPase; Cache = small ligand binding domain; EAL = phosphodiesterase; GAF = cGMP-binding domain; GGDEF = diguanylate cyclase; HK = histidine kinase domain; HTH = helix-turn-helix; PAS = sensor domain (may include a PAS/PAC arrangement); PP = phosphatase; RR = response regulator domain; SP = signal peptide; TM = transmembrane region. <br>
</h4>
</div>
<div class="modal-body"><img style="max-width:100%" src="1414-431X-bjmbr-37-04-459-477-g5500i05.jpg"></div>
</div></div></div>
<div class="modal fade ModalFigs" id="ModalFigfig6" tabindex="-1" role="dialog" aria-hidden="true"><div class="modal-dialog modal-lg"><div class="modal-content">
<div class="modal-header">
<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">×</span><span class="sr-only">Close</span></button><a class="link-newWindow showTooltip" target="_blank" data-placement="left" title="Open new window" href="1414-431X-bjmbr-37-04-459-477-g5500i06.jpg"><span class="sci-ico-newWindow"></span></a><h4 class="modal-title">
<span class="sci-ico-fileFigure"></span>Figure 6.    Model of leptospiral membrane architecture. Leptospires have two membranes, an outer membrane (OM) and a cytoplasmic or inner membrane (IM). As in Gram-positive bacteria, the peptidoglycan (PG) cell wall is closely associated with the IM. The leptospiral surface is dominated by lipopolysaccharide (LPS) carbohydrate side chains. Subsurface proteins include the cytoplasmic protein, GroEL, and the periplasmic endoflagella (EF). The IM contains lipoproteins such as LipL31 and transmembrane proteins such as signal peptidase (SP) and ImpL63. The OM contains lipoproteins including LipL41 and LipL36, and transmembrane proteins including the porin, OmpL1. Genomic sequence analysis reveals several novel types of outer membrane proteins (OMPs), including TonB-dependent OMPs involved in nutrient acquisition. BtuB is an example of a TonB-dependent OMP involved in uptake of vitamin B<sub>12</sub>. The type I efflux system is represented by TolC, which forms a complex with ATP-binding cassette (ABC) transporters to export hemolysins and other cytoplasmic components. The leptospiral genome also contains genes involved in a three-component efflux system consisting of an outer membrane factor (OMF), membrane fusion protein (MFP), and an inner membrane transporter, in this case CzcA, which is involved in heavy metal detoxification. <br>
</h4>
</div>
<div class="modal-body"><img style="max-width:100%" src="1414-431X-bjmbr-37-04-459-477-g5500i06.jpg"></div>
</div></div></div>
<div class="modal fade ModalFigs" id="ModalFigfig7" tabindex="-1" role="dialog" aria-hidden="true"><div class="modal-dialog modal-lg"><div class="modal-content">
<div class="modal-header">
<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">×</span><span class="sr-only">Close</span></button><a class="link-newWindow showTooltip" target="_blank" data-placement="left" title="Open new window" href="1414-431X-bjmbr-37-04-459-477-g5500i07.jpg"><span class="sci-ico-newWindow"></span></a><h4 class="modal-title">
<span class="sci-ico-fileFigure"></span>Figure 7.    Lipopolysaccharide region of <i>Leptospira interrogans.</i> The 119-kb region of <i>L. interrogans</i> serovar Copenhageni genome with 108 predicted coding sequences transcribed in the same orientation. Orange arrows indicate predicted genes encoding polysaccharide biosynthesis-related proteins. Shaded regions indicate the homologous genes from <i>L. borgpetersenii</i> serovar Hardjobovis (53). The region containing the predicted coding sequences, indicated by the letters <i>F, E, A, B, D, C</i>, was first indicated as the <i>rfb</i> genes related to rhamnose biosynthesis in <i>L. interrogans</i> Copenhageni (54). <br>
</h4>
</div>
<div class="modal-body"><img style="max-width:100%" src="1414-431X-bjmbr-37-04-459-477-g5500i07.jpg"></div>
</div></div></div>
<div class="modal fade ModalTables" id="ModalTablet01" tabindex="-1" role="dialog" aria-hidden="true"><div class="modal-dialog modal-lg"><div class="modal-content">
<div class="modal-header">
<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">×</span><span class="sr-only">Close</span></button><a class="link-newWindow showTooltip" href="1414-431X-bjmbr-37-04-459-477-g5500t01.gif" target="_blank" data-placement="left" title="Open new window"><span class="sci-ico-newWindow"></span></a><h4 class="modal-title">
<span class="sci-ico-fileTable"></span><strong></strong><br>
</h4>
</div>
<div class="modal-body"><img style="max-width:100%" src="1414-431X-bjmbr-37-04-459-477-g5500t01.gif"></div>
<div class="modal-footer"></div>
</div></div></div>
<div class="modal fade ModalTables" id="ModalTablet02" tabindex="-1" role="dialog" aria-hidden="true"><div class="modal-dialog modal-lg"><div class="modal-content">
<div class="modal-header">
<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">×</span><span class="sr-only">Close</span></button><a class="link-newWindow showTooltip" href="1414-431X-bjmbr-37-04-459-477-g5500t02.gif" target="_blank" data-placement="left" title="Open new window"><span class="sci-ico-newWindow"></span></a><h4 class="modal-title">
<span class="sci-ico-fileTable"></span><strong></strong><br>
</h4>
</div>
<div class="modal-body"><img style="max-width:100%" src="1414-431X-bjmbr-37-04-459-477-g5500t02.gif"></div>
<div class="modal-footer"></div>
</div></div></div>
<div class="modal fade ModalDefault" id="ModalArticles" tabindex="-1" role="dialog" aria-hidden="true"><div class="modal-dialog"><div class="modal-content">
<div class="modal-header">
<button type="button" class="close" data-dismiss="modal"><span aria-hidden="true">×</span><span class="sr-only">Close</span></button><h4 class="modal-title">How to cite</h4>
</div>
<div class="modal-body">
<p id="citation"></p>
<input id="citationCut" type="text" value=""><a class="copyLink" data-clipboard-target="#citationCut"><span class="glyphBtn copyIcon"></span>copy</a>
</div>
</div></div></div>
<script type="text/javascript">
            function currentDate() {
            var today = new Date();
            var months = ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December']
            today.setTime(today.getTime());
            return today.getDate() + " " + months[today.getMonth()] + " " + today.getFullYear();
            }
            var citation = 'Nascimento, A.L.T.O. et al. Genome features of Leptospira interrogans serovar Copenhageni. Brazilian Journal of Medical and Biological Research [online]. 2004, v. 37, n. 4 [Accessed CURRENTDATE], pp. 459-477. Available from: &lt;https://doi.org/10.1590/S0100-879X2004000400003&gt;. Epub 22 Apr 2004. ISSN 1414-431X. https://doi.org/10.1590/S0100-879X2004000400003.'.replace('CURRENTDATE', currentDate());
            document.getElementById('citation').innerHTML = citation;
            document.getElementById('citationCut').value = citation.replace('&lt;', '<').replace('&gt;', ">");
        </script>
</div>
<ul class="floatingMenu fm-slidein" data-fm-toogle="hover"><li class="fm-wrap">
<a href="javascript:;" class="fm-button-main"><span class="sci-ico-floatingMenuDefault glyphFloatMenu"></span><span class="sci-ico-floatingMenuClose glyphFloatMenu"></span></a><ul class="fm-list">
<li><a class="fm-button-child" data-fm-label="Figures | Tables" data-toggle="modal" data-target="#ModalTablesFigures"><span class="sci-ico-figures glyphFloatMenu"></span></a></li>
<li><a class="fm-button-child" data-toggle="modal" data-target="#ModalArticles" data-fm-label="How to
                                          cite"><span class="sci-ico-citation glyphFloatMenu"></span></a></li>
</ul>
</li></ul>
<script src="/Users/roberta.takenaka/github.com/scieloorg/packtools/packtools/catalogs/htmlgenerator/static/scielo-article-standalone-min.js"></script>
</body>
</html>
